"use strict";(self.webpackChunklotfollahi_com=self.webpackChunklotfollahi_com||[]).push([[6349],{3905:(e,t,a)=>{a.d(t,{Zo:()=>c,kt:()=>m});var l=a(7294);function n(e,t,a){return t in e?Object.defineProperty(e,t,{value:a,enumerable:!0,configurable:!0,writable:!0}):e[t]=a,e}function i(e,t){var a=Object.keys(e);if(Object.getOwnPropertySymbols){var l=Object.getOwnPropertySymbols(e);t&&(l=l.filter((function(t){return Object.getOwnPropertyDescriptor(e,t).enumerable}))),a.push.apply(a,l)}return a}function r(e){for(var t=1;t<arguments.length;t++){var a=null!=arguments[t]?arguments[t]:{};t%2?i(Object(a),!0).forEach((function(t){n(e,t,a[t])})):Object.getOwnPropertyDescriptors?Object.defineProperties(e,Object.getOwnPropertyDescriptors(a)):i(Object(a)).forEach((function(t){Object.defineProperty(e,t,Object.getOwnPropertyDescriptor(a,t))}))}return e}function o(e,t){if(null==e)return{};var a,l,n=function(e,t){if(null==e)return{};var a,l,n={},i=Object.keys(e);for(l=0;l<i.length;l++)a=i[l],t.indexOf(a)>=0||(n[a]=e[a]);return n}(e,t);if(Object.getOwnPropertySymbols){var i=Object.getOwnPropertySymbols(e);for(l=0;l<i.length;l++)a=i[l],t.indexOf(a)>=0||Object.prototype.propertyIsEnumerable.call(e,a)&&(n[a]=e[a])}return n}var s=l.createContext({}),p=function(e){var t=l.useContext(s),a=t;return e&&(a="function"==typeof e?e(t):r(r({},t),e)),a},c=function(e){var t=p(e.components);return l.createElement(s.Provider,{value:t},e.children)},u="mdxType",h={inlineCode:"code",wrapper:function(e){var t=e.children;return l.createElement(l.Fragment,{},t)}},g=l.forwardRef((function(e,t){var a=e.components,n=e.mdxType,i=e.originalType,s=e.parentName,c=o(e,["components","mdxType","originalType","parentName"]),u=p(a),g=n,m=u["".concat(s,".").concat(g)]||u[g]||h[g]||i;return a?l.createElement(m,r(r({ref:t},c),{},{components:a})):l.createElement(m,r({ref:t},c))}));function m(e,t){var a=arguments,n=t&&t.mdxType;if("string"==typeof e||n){var i=a.length,r=new Array(i);r[0]=g;var o={};for(var s in t)hasOwnProperty.call(t,s)&&(o[s]=t[s]);o.originalType=e,o[u]="string"==typeof e?e:n,r[1]=o;for(var p=2;p<i;p++)r[p]=a[p];return l.createElement.apply(null,r)}return l.createElement.apply(null,a)}g.displayName="MDXCreateElement"},4449:(e,t,a)=>{a.r(t),a.d(t,{contentTitle:()=>r,default:()=>u,frontMatter:()=>i,metadata:()=>o,toc:()=>s});var l=a(7462),n=(a(7294),a(3905));const i={},r=void 0,o={type:"mdx",permalink:"/publications",source:"@site/src/pages/publications.md",description:"- See a full list of papers on Google Scholar.",frontMatter:{}},s=[{value:"2023",id:"2023",level:3},{value:"An integrated single-cell reference atlas of the human endometrium.",id:"an-integrated-single-cell-reference-atlas-of-the-human-endometrium",level:5},{value:"Population-level integration of single-cell datasets enables multi-scale analysis across samples.",id:"population-level-integration-of-single-cell-datasets-enables-multi-scale-analysis-across-samples",level:5},{value:"Predicting cell morphological responses to perturbations using generative modeling",id:"predicting-cell-morphological-responses-to-perturbations-using-generative-modeling",level:5},{value:"An integrated cell atlas of the human lung in health and disease",id:"an-integrated-cell-atlas-of-the-human-lung-in-health-and-disease",level:5},{value:"Predicting cellular responses to complex perturbations in high\u2010throughput screens",id:"predicting-cellular-responses-to-complex-perturbations-in-highthroughput-screens",level:5},{value:"The scverse project provides a computational ecosystem for single-cell omics data analysis",id:"the-scverse-project-provides-a-computational-ecosystem-for-single-cell-omics-data-analysis",level:5},{value:"Best practices for single-cell analysis across modalities",id:"best-practices-for-single-cell-analysis-across-modalities",level:5},{value:"Biologically Informed Deep Learning to Query Gene Programs in Single-Cell Atlases",id:"biologically-informed-deep-learning-to-query-gene-programs-in-single-cell-atlases",level:5},{value:"Mapping cells to gene programs",id:"mapping-cells-to-gene-programs",level:5},{value:"Single-cell RNA sequencing uncovers heterogeneous transcriptional signatures in tumor-infiltrated dendritic cells in prostate cancer. Heliyon, 9(5), e15694.",id:"single-cell-rna-sequencing-uncovers-heterogeneous-transcriptional-signatures-in-tumor-infiltrated-dendritic-cells-in-prostate-cancer-heliyon-95-e15694",level:4},{value:"2022",id:"2022",level:3},{value:"Predicting single-cell perturbation responses for unseen drugs",id:"predicting-single-cell-perturbation-responses-for-unseen-drugs",level:5},{value:"Modelling method using a conditional variational autoencoder",id:"modelling-method-using-a-conditional-variational-autoencoder",level:5},{value:"Deep Learning in Spatially Resolved Transcriptomics: A Comprehensive Technical View",id:"deep-learning-in-spatially-resolved-transcriptomics-a-comprehensive-technical-view",level:5},{value:"Single-cell reference mapping to construct and extend cell type hierarchies",id:"single-cell-reference-mapping-to-construct-and-extend-cell-type-hierarchies",level:5},{value:"Squidpy: a scalable framework for spatial omics analysis",id:"squidpy-a-scalable-framework-for-spatial-omics-analysis",level:5},{value:"A Python library for probabilistic analysis of single-cell omics data",id:"a-python-library-for-probabilistic-analysis-of-single-cell-omics-data",level:5},{value:"Population-level integration of single-cell datasets enables multi-scale analysis across samples.",id:"population-level-integration-of-single-cell-datasets-enables-multi-scale-analysis-across-samples-1",level:5},{value:"An integrated cell atlas of the human lung in health and disease",id:"an-integrated-cell-atlas-of-the-human-lung-in-health-and-disease-1",level:5},{value:"Deep generative modeling of transcriptional dynamics for RNA velocity analysis in single cells.",id:"deep-generative-modeling-of-transcriptional-dynamics-for-rna-velocity-analysis-in-single-cells",level:5},{value:"MultiCPA: Multimodal Compositional Perturbation Autoencoder",id:"multicpa-multimodal-compositional-perturbation-autoencoder",level:5},{value:"Continual single-cell architecture surgery for reference mapping.",id:"continual-single-cell-architecture-surgery-for-reference-mapping",level:5},{value:"Mapping single-cell data to reference atlases by transfer learning",id:"mapping-single-cell-data-to-reference-atlases-by-transfer-learning",level:5},{value:"2021",id:"2021",level:3},{value:"Multigrate: Single-Cell Multi-Omic Data Integration",id:"multigrate-single-cell-multi-omic-data-integration",level:5},{value:"Integrating T-cell receptor and transcriptome for large-scale single-cell immune profiling analysis",id:"integrating-t-cell-receptor-and-transcriptome-for-large-scale-single-cell-immune-profiling-analysis",level:5},{value:"Machine learning for perturbational single-cell omics",id:"machine-learning-for-perturbational-single-cell-omics",level:5},{value:"2020",id:"2020",level:3},{value:"Out-of-distribution prediction with disentangled representations for single-cell RNA sequencing data",id:"out-of-distribution-prediction-with-disentangled-representations-for-single-cell-rna-sequencing-data",level:5},{value:"Conditional out-of-distribution generation for unpaired data using transfer VAE",id:"conditional-out-of-distribution-generation-for-unpaired-data-using-transfer-vae",level:4},{value:"2019",id:"2019",level:3},{value:"Deep packet: a novel approach for encrypted traffic classification using deep learning",id:"deep-packet-a-novel-approach-for-encrypted-traffic-classification-using-deep-learning",level:5}],p={toc:s},c="wrapper";function u(e){let{components:t,...a}=e;return(0,n.kt)(c,(0,l.Z)({},p,a,{components:t,mdxType:"MDXLayout"}),(0,n.kt)("ul",null,(0,n.kt)("li",{parentName:"ul"},"See a full list of papers on ",(0,n.kt)("strong",{parentName:"li"},(0,n.kt)("a",{parentName:"strong",href:"https://scholar.google.com/citations?user=NXhouUcAAAAJ&hl=en"},"Google Scholar")),"."),(0,n.kt)("li",{parentName:"ul"},"Co-first and co-last authors are indicated by + and *, respectively.")),(0,n.kt)("h3",{id:"2023"},"2023"),(0,n.kt)("h5",{id:"an-integrated-single-cell-reference-atlas-of-the-human-endometrium"},"An integrated single-cell reference atlas of the human endometrium."),(0,n.kt)("p",null,"Mare\u010dkov\xe1, M., Garcia-Alonso, L., Moullet, M., Lorenzi, V., Petryszak, R., Sancho-Serra, C., ",(0,n.kt)("u",null,"Lotfollahi,M","*")," ... &\nVento-Tormo, R. (2023)."),(0,n.kt)("p",null,(0,n.kt)("a",{parentName:"p",href:"https://www.biorxiv.org/content/10.1101/2023.11.03.564728v1.abstract"},"[bioRxiv (2023)]"),"."),(0,n.kt)("h5",{id:"population-level-integration-of-single-cell-datasets-enables-multi-scale-analysis-across-samples"},"Population-level integration of single-cell datasets enables multi-scale analysis across samples."),(0,n.kt)("p",null,"De Donno, C., Hediyeh-Zadeh, S., Wagenstetter, M., Moinfar, A. A., Zappia, L., "),(0,n.kt)("u",null,"Lotfollahi, M.* "),", & Theis, F. J*.",(0,n.kt)("p",null,(0,n.kt)("a",{parentName:"p",href:"https://github.com/theislab/scArches"},"[code]"),",\n",(0,n.kt)("a",{parentName:"p",href:"https://www.nature.com/articles/s41592-023-02035-2"},"[Nature Methods (2023)]"),"."),(0,n.kt)("hr",null),(0,n.kt)("h5",{id:"predicting-cell-morphological-responses-to-perturbations-using-generative-modeling"},"Predicting cell morphological responses to perturbations using generative modeling"),(0,n.kt)("p",null,"Palma, A., Theis, F. J.","*",", ",(0,n.kt)("u",null,"Lotfollahi,M","*"),"."),(0,n.kt)("p",null,(0,n.kt)("a",{parentName:"p",href:"https://www.biorxiv.org/content/10.1101/2023.07.17.549216v1"},"[bioRxiv (2023)]"),".\n",(0,n.kt)("a",{parentName:"p",href:"https://github.com/theislab/impa"},"[code]"),"."),(0,n.kt)("hr",null),(0,n.kt)("h5",{id:"an-integrated-cell-atlas-of-the-human-lung-in-health-and-disease"},"An integrated cell atlas of the human lung in health and disease"),(0,n.kt)("p",null,"Sikkema, L., Strobl, D., Zappia, L., Madissoon, E., Markov, N. S., Zaragosi, L., Ansari, M., Arguel, M., Apperloo, L., B\xe9cavin, C., Berg, M., Chichelnitskiy, E., Chung, M.,\nCollin, A., Gay, A. C. A., Hooshiar Kashani, B., Jain, M., Kapellos, T., Kole, T. M., \u2026,",(0,n.kt)("u",null,"Lotfollahi. M"),",...,Theis, F."),(0,n.kt)("p",null,(0,n.kt)("a",{parentName:"p",href:"https://www.nature.com/articles/s41591-023-02327-2"},"[Nature Medicine (2023)]"),".\n",(0,n.kt)("a",{parentName:"p",href:"https://github.com/LungCellAtlas/HLCA_reproducibility"},"[code]"),",\n",(0,n.kt)("a",{parentName:"p",href:"https://docs.scarches.org/en/latest/hlca_map_classify.html"},"[Mapping data to HLCA using scArhes]"),","),(0,n.kt)("hr",null),(0,n.kt)("h5",{id:"predicting-cellular-responses-to-complex-perturbations-in-highthroughput-screens"},"Predicting cellular responses to complex perturbations in high\u2010throughput screens"),(0,n.kt)("u",null,"Lotfollahi, M+."),", Klimovskaia Susmelj+, A., De Donno, C+., Hetzel, L., Ji, Y., Ibarra, I. L., ... & Theis, F. J.",(0,n.kt)("p",null,(0,n.kt)("a",{parentName:"p",href:"https://www.embopress.org/doi/full/10.15252/msb.202211517"},"[Molecular Systems Biology (2023)]"),",\n",(0,n.kt)("a",{parentName:"p",href:"https://github.com/facebookresearch/CPA"},"[code]"),",\n",(0,n.kt)("a",{parentName:"p",href:"https://ai.facebook.com/blog/ai-predicts-effective-drug-combinations-to-fight-complex-diseases-faster"},"[Facebook AI blogpost]"),",\n",(0,n.kt)("a",{parentName:"p",href:"https://www.stateof.ai/2021"},"[state of AI report 2021]"),",\n",(0,n.kt)("a",{parentName:"p",href:"https://www.embopress.org/loi/17444292"},"[featured cover]"),"."),(0,n.kt)("hr",null),(0,n.kt)("h5",{id:"the-scverse-project-provides-a-computational-ecosystem-for-single-cell-omics-data-analysis"},"The scverse project provides a computational ecosystem for single-cell omics data analysis"),(0,n.kt)("p",null,"Virshup+ ",(0,n.kt)("em",{parentName:"p"},"et al"),". "),(0,n.kt)("p",null,(0,n.kt)("a",{parentName:"p",href:"https://www.embopress.org/doi/full/10.15252/msb.202211517"},"[Nature biotechnology (2023)]"),",\n",(0,n.kt)("a",{parentName:"p",href:"https://scverse.org/"},"[community]"),"."),(0,n.kt)("hr",null),(0,n.kt)("h5",{id:"best-practices-for-single-cell-analysis-across-modalities"},"Best practices for single-cell analysis across modalities"),(0,n.kt)("p",null,"Heumos+ ",(0,n.kt)("em",{parentName:"p"},"et al"),". "),(0,n.kt)("p",null,(0,n.kt)("a",{parentName:"p",href:"https://www.embopress.org/doi/full/10.15252/msb.202211517"},"[Nature Reviews Genetics (2023)]"),",\n",(0,n.kt)("a",{parentName:"p",href:"https://www.sc-best-practices.org/preamble.html"},"[code)]"),"."),(0,n.kt)("hr",null),(0,n.kt)("h5",{id:"biologically-informed-deep-learning-to-query-gene-programs-in-single-cell-atlases"},"Biologically Informed Deep Learning to Query Gene Programs in Single-Cell Atlases"),(0,n.kt)("u",null,"Lotfollahi M"),", M+., Rybakov, S+., Hrovatin, K., Hediyeh-Zadeh, S., Talavera-L\xf3pez, C., Misharin, A. V., & Theis, F. J.",(0,n.kt)("p",null,(0,n.kt)("a",{parentName:"p",href:"https://icml-compbio.github.io/icml-website-2021/2021/papers/WCBICML2021_paper_45.pdf"},"[Nature Cell Biology (2023)]"),",\n",(0,n.kt)("a",{parentName:"p",href:"https://github.com/theislab/scArches"},"[code]"),"."),(0,n.kt)("hr",null),(0,n.kt)("h5",{id:"mapping-cells-to-gene-programs"},"Mapping cells to gene programs"),(0,n.kt)("u",null,"Lotfollahi M"),", M+., Rybakov, S+., Hrovatin, K., Hediyeh-Zadeh, S., Talavera-L\xf3pez, C., Misharin, A. V., & Theis, F. J.",(0,n.kt)("p",null,(0,n.kt)("a",{parentName:"p",href:"https://protocolexchange.researchsquare.com/article/pex-2092/v1"},"[Protocole exchange (2023)]"),",\n",(0,n.kt)("a",{parentName:"p",href:"https://github.com/theislab/scArches"},"[code]"),"."),(0,n.kt)("hr",null),(0,n.kt)("h4",{id:"single-cell-rna-sequencing-uncovers-heterogeneous-transcriptional-signatures-in-tumor-infiltrated-dendritic-cells-in-prostate-cancer-heliyon-95-e15694"},"Single-cell RNA sequencing uncovers heterogeneous transcriptional signatures in tumor-infiltrated dendritic cells in prostate cancer. Heliyon, 9(5), e15694."),(0,n.kt)("p",null,"Feriz, A. M., Khosrojerdi, A., ",(0,n.kt)("u",null,"Lotfollahi, M."),", Shamsaki, N., GhasemiGol, M., HosseiniGol, E., Fereidouni, M., Rohban, M. H., Sebzari, A. R., Saghafi, S., Leone, P., Silvestris, N., Safarpour, H., & Racanelli, V. (2023). "),(0,n.kt)("p",null,(0,n.kt)("a",{parentName:"p",href:"https://www.cell.com/heliyon/pdf/S2405-8440(23)02901-8.pdf"},"[Heliyon(2023)]"),"."),(0,n.kt)("h3",{id:"2022"},"2022"),(0,n.kt)("h5",{id:"predicting-single-cell-perturbation-responses-for-unseen-drugs"},"Predicting single-cell perturbation responses for unseen drugs"),(0,n.kt)("p",null,"Hetzel, L., B\xf6hm, S., Kilbertus, N., G\xfcnnemann, S., ",(0,n.kt)("u",null,"Lotfollahi, M."),", & Theis, F. (2022)."),(0,n.kt)("p",null,(0,n.kt)("a",{parentName:"p",href:"https://nips.cc/Conferences/2022/ScheduleMultitrack?event=53227"},"[NeurIPS (2022)]"),",\n",(0,n.kt)("a",{parentName:"p",href:"https://github.com/theislab/chemcpa"},"[code]"),"."),(0,n.kt)("hr",null),(0,n.kt)("h5",{id:"modelling-method-using-a-conditional-variational-autoencoder"},"Modelling method using a conditional variational autoencoder"),(0,n.kt)("p",null,"Theis, F. J.,",(0,n.kt)("u",null,"Lotfollahi, M."),", Wolf, F. A. "),(0,n.kt)("p",null,(0,n.kt)("a",{parentName:"p",href:"https://patents.google.com/patent/US20220383985A1/en"},"US Patent"),"."),(0,n.kt)("hr",null),(0,n.kt)("h5",{id:"deep-learning-in-spatially-resolved-transcriptomics-a-comprehensive-technical-view"},"Deep Learning in Spatially Resolved Transcriptomics: A Comprehensive Technical View"),(0,n.kt)("p",null,"Nasab, R. Z., Ghamsari, M. R. E., Argha, A., Macphillamy, C., Beheshti, A., Alizadehsani, R., Lovell, N. H.,"),(0,n.kt)("u",null,"Lotfollahi, M."),", & Alinejad-Rokny, H.",(0,n.kt)("p",null,(0,n.kt)("a",{parentName:"p",href:"https://arxiv.org/abs/2210.04453"},"arXiv"),"."),(0,n.kt)("hr",null),(0,n.kt)("h5",{id:"single-cell-reference-mapping-to-construct-and-extend-cell-type-hierarchies"},"Single-cell reference mapping to construct and extend cell type hierarchies"),(0,n.kt)("p",null,"Michielsen,L+.,",(0,n.kt)("u",null,"Lotfollahi, M.+"),", Strobl, D., Sikkema, L., Reinders, M. J. T., Theis, F. J.,Mahfouz, A. "),(0,n.kt)("p",null,(0,n.kt)("a",{parentName:"p",href:"https://www.biorxiv.org/content/10.1101/2022.07.07.499109v1"},"[BioRxv (2022)]"),",\n",(0,n.kt)("a",{parentName:"p",href:"https://github.com/theislab/scarches"},"[code]"),"."),(0,n.kt)("hr",null),(0,n.kt)("h5",{id:"squidpy-a-scalable-framework-for-spatial-omics-analysis"},"Squidpy: a scalable framework for spatial omics analysis"),(0,n.kt)("p",null,"Palla, G., Spitzer, H., Klein, M., Fischer, D., Schaar, A. C., Kuemmerle, L. B., Rybakov, S.,\nIbarra, I. L., Holmberg, O., Virshup, I., ",(0,n.kt)("u",null,"Lotfollahi, M."),", Richter, S. & Theis, F. J. "),(0,n.kt)("p",null,(0,n.kt)("a",{parentName:"p",href:"https://www.nature.com/articles/s41592-021-01358-2"},"[Nature Methods (2022)]"),",\n",(0,n.kt)("a",{parentName:"p",href:"https://github.com/theislab/squidpy"},"[code]"),"."),(0,n.kt)("hr",null),(0,n.kt)("h5",{id:"a-python-library-for-probabilistic-analysis-of-single-cell-omics-data"},"A Python library for probabilistic analysis of single-cell omics data"),(0,n.kt)("p",null,"Gayoso, A.+, Lopez, R.+, Xing, G., Boyeau+, P., Valiollah Pour Amiri, V., Hong, J., Wu, K., Jayasuriya, M., Mehlman, E., Langevin, M., Liu, Y., Samaran, J.,\nMisrachi, G., Nazaret, A., Clivio, O., Xu, C., Ashuach, T., Gabitto, M., ",(0,n.kt)("u",null,"Lotfollahi, M."),", \u2026 Yosef, N. "),(0,n.kt)("p",null,(0,n.kt)("a",{parentName:"p",href:"https://www.nature.com/articles/s41587-021-01206-w"},"[Nature Biotechnology (2022)]"),",\n",(0,n.kt)("a",{parentName:"p",href:"https://scvi-tools.org/"},"[code]"),"."),(0,n.kt)("hr",null),(0,n.kt)("h5",{id:"population-level-integration-of-single-cell-datasets-enables-multi-scale-analysis-across-samples-1"},"Population-level integration of single-cell datasets enables multi-scale analysis across samples."),(0,n.kt)("p",null,"De Donno, C., Hediyeh-Zadeh, S., Wagenstetter, M., Moinfar, A. A., Zappia, L., "),(0,n.kt)("u",null,"Lotfollahi, M.* "),", & Theis, F. J*.",(0,n.kt)("p",null,(0,n.kt)("a",{parentName:"p",href:"https://github.com/theislab/scArches"},"[code]"),",\n",(0,n.kt)("a",{parentName:"p",href:"https://www.biorxiv.org/content/10.1101/2022.11.28.517803v1"},"[bioRxv (2022)]"),"."),(0,n.kt)("hr",null),(0,n.kt)("h5",{id:"an-integrated-cell-atlas-of-the-human-lung-in-health-and-disease-1"},"An integrated cell atlas of the human lung in health and disease"),(0,n.kt)("p",null,"Sikkema, L., Strobl, D., Zappia, L., Madissoon, E., Markov, N. S., Zaragosi, L., Ansari, M., Arguel, M., Apperloo, L., B\xe9cavin, C., Berg, M., Chichelnitskiy, E., Chung, M.,\nCollin, A., Gay, A. C. A., Hooshiar Kashani, B., Jain, M., Kapellos, T., Kole, T. M., \u2026,",(0,n.kt)("u",null,"Lotfollahi. M"),",...,Theis, F."),(0,n.kt)("p",null,(0,n.kt)("a",{parentName:"p",href:"https://github.com/LungCellAtlas/HLCA_reproducibility"},"[code]"),","),(0,n.kt)("p",null,(0,n.kt)("a",{parentName:"p",href:"https://www.biorxiv.org/content/10.1101/2022.03.10.483747v1"},"[bioRxv (2022)]"),"."),(0,n.kt)("hr",null),(0,n.kt)("h5",{id:"deep-generative-modeling-of-transcriptional-dynamics-for-rna-velocity-analysis-in-single-cells"},"Deep generative modeling of transcriptional dynamics for RNA velocity analysis in single cells."),(0,n.kt)("p",null,"Gayoso, A., Weiler, P.,",(0,n.kt)("u",null," Lotfollahi, M."),", Klein, D., Hong, J., Streets, A., Theis, F. J., & Yosef, N."),(0,n.kt)("p",null,(0,n.kt)("a",{parentName:"p",href:"https://github.com/YosefLab/velovi"},"[code]"),","),(0,n.kt)("p",null,(0,n.kt)("a",{parentName:"p",href:"https://www.biorxiv.org/content/10.1101/2022.03.10.483747v1"},"[bioRxv (2022)]"),"."),(0,n.kt)("hr",null),(0,n.kt)("h5",{id:"multicpa-multimodal-compositional-perturbation-autoencoder"},"MultiCPA: Multimodal Compositional Perturbation Autoencoder"),(0,n.kt)("p",null,"Inecik, K., Uhlmann, A.,"),(0,n.kt)("u",null,"Lotfollahi,M.* "),", Theis,F*.",(0,n.kt)("p",null,(0,n.kt)("a",{parentName:"p",href:"https://icml-compbio.github.io/icml-website-2022/"},"[ICML Workshop on Computational Biology (WCB) 2022]"),",\n",(0,n.kt)("a",{parentName:"p",href:"https://www.biorxiv.org/content/10.1101/2022.07.08.499049v1"},"[bioRxv]"),",\n",(0,n.kt)("a",{parentName:"p",href:"https://github.com/theislab/multicpa"},"[code]"),"."),(0,n.kt)("hr",null),(0,n.kt)("h5",{id:"continual-single-cell-architecture-surgery-for-reference-mapping"},"Continual single-cell architecture surgery for reference mapping."),(0,n.kt)("p",null,"Hediyeh-zadeh, S.,"),(0,n.kt)("u",null,"Lotfollahi, M.* "),", & Theis, F*.",(0,n.kt)("p",null,(0,n.kt)("a",{parentName:"p",href:"https://icml-compbio.github.io/2022/papers/WCBICML2022_paper_68.pdf"},"[ICML Workshop on Computational Biology (WCB) 2022]"),","),(0,n.kt)("hr",null),(0,n.kt)("h5",{id:"mapping-single-cell-data-to-reference-atlases-by-transfer-learning"},"Mapping single-cell data to reference atlases by transfer learning"),(0,n.kt)("u",null,"Lotfollahi, M."),", Naghipourfar, M., Luecken, M. D., Khajavi, M., B\xfcttner, M., Wagenstetter, M., Avsec, \u017d., Gayoso, A., Yosef, N., Interlandi, M. & Others.",(0,n.kt)("p",null,(0,n.kt)("a",{parentName:"p",href:"https://www.nature.com/articles/s41587-021-01001-7"},"[Nature Biotechnology (2022)]"),",\n",(0,n.kt)("a",{parentName:"p",href:"https://github.com/theislab/scarches"},"[code]"),",\n",(0,n.kt)("a",{parentName:"p",href:"https://www.mdsi.tum.de/en/mdsi/latest-info/news/full-text/article/awardees-of-the-mdsi-best-paper-of-the-year-award/"},"[MDSI best paper award]"),",\n",(0,n.kt)("a",{parentName:"p",href:"https://www.nature.com/nbt/volumes/40/issues/1"},"[featured cover in Nature Biotechnology]"),"."),(0,n.kt)("h3",{id:"2021"},"2021"),(0,n.kt)("h5",{id:"multigrate-single-cell-multi-omic-data-integration"},"Multigrate: Single-Cell Multi-Omic Data Integration"),(0,n.kt)("u",null,"Lotfollahi. M+"),"., Litinetskaya, A+. and Theis, F. J.",(0,n.kt)("p",null,(0,n.kt)("a",{parentName:"p",href:"https://icml-compbio.github.io/icml-website-2021/"},"[contributed talk Award at ICML Workshop on Computational Biology 2021]"),",\n",(0,n.kt)("a",{parentName:"p",href:"https://github.com/theislab/multigrate"},"[code]"),",\n",(0,n.kt)("a",{parentName:"p",href:"https://www.biorxiv.org/content/10.1101/2022.03.16.484643v1"},"[bioRxv (2022)]"),"."),(0,n.kt)("hr",null),(0,n.kt)("h5",{id:"integrating-t-cell-receptor-and-transcriptome-for-large-scale-single-cell-immune-profiling-analysis"},"Integrating T-cell receptor and transcriptome for large-scale single-cell immune profiling analysis"),(0,n.kt)("p",null,"Drost, F., An, Y., Dratva, L. M., Lindeboom, R. G. H., Haniffa, M., Teichmann, S. A., Theis, F., ",(0,n.kt)("u",null,"Lotfollahi, M.","*"),", Schubert, B","*","."),(0,n.kt)("p",null,(0,n.kt)("a",{parentName:"p",href:"https://icml-compbio.github.io/icml-website-2021/2021/papers/WCBICML2021_paper_45.pdf"},"[ICML Workshop on Computational Biology 2021]"),",\n",(0,n.kt)("a",{parentName:"p",href:"https://github.com/SchubertLab/mvTCR"},"[code]"),",\n",(0,n.kt)("a",{parentName:"p",href:"https://www.biorxiv.org/content/10.1101/2021.06.24.449733v2.abstract"},"[bioRxv (2021)]"),"."),(0,n.kt)("hr",null),(0,n.kt)("h5",{id:"machine-learning-for-perturbational-single-cell-omics"},"Machine learning for perturbational single-cell omics"),(0,n.kt)("p",null,"Ji, Y.,",(0,n.kt)("u",null," Lotfollahi, M."),", Wolf, F. A. & Theis, F. J."),(0,n.kt)("p",null,(0,n.kt)("a",{parentName:"p",href:"https://doi.org/10.1016/j.cels.2021.05.016"},"[Cell Systems(2021)]"),",\n",(0,n.kt)("a",{parentName:"p",href:"https://github.com/theislab/sc-pert"},"[data resource]"),"."),(0,n.kt)("h3",{id:"2020"},"2020"),(0,n.kt)("h5",{id:"out-of-distribution-prediction-with-disentangled-representations-for-single-cell-rna-sequencing-data"},"Out-of-distribution prediction with disentangled representations for single-cell RNA sequencing data"),(0,n.kt)("u",null,"Lotfollahi, M.+"),", Dony, L.+, Agarwala, H.+, & Theis, F. J.",(0,n.kt)("p",null,(0,n.kt)("a",{parentName:"p",href:"https://icml-compbio.github.io/icml-website-2020/"},"[ICML Workshop on Computational Biology (WCB) 2020]"),",\n",(0,n.kt)("a",{parentName:"p",href:"https://slideslive.com/38931310/outofdistribution-prediction-with-disentangled-representations-for-scrnaseq-data"},"[spotlight talk ICML WCB 2020]"),",\n",(0,n.kt)("a",{parentName:"p",href:"https://github.com/theislab/disent"},"[code]")),(0,n.kt)("hr",null),(0,n.kt)("h4",{id:"conditional-out-of-distribution-generation-for-unpaired-data-using-transfer-vae"},"Conditional out-of-distribution generation for unpaired data using transfer VAE"),(0,n.kt)("u",null,"Lotfollahi, M."),", Naghipourfar, M., Theis, F. J. & Wolf, F. A.",(0,n.kt)("p",null,(0,n.kt)("a",{parentName:"p",href:"https://doi.org/10.1093/bioinformatics/btaa800"},"[Bioinformatics(2020)]"),",\n",(0,n.kt)("a",{parentName:"p",href:"https://github.com/theislab/trvae"},"[","code","]"),",\n",(0,n.kt)("a",{parentName:"p",href:"https://eccb2020.info/"},"[","talk at ECCB 2020 (21.18% accept rate)","]"),"."),(0,n.kt)("hr",null),(0,n.kt)("p",null,(0,n.kt)("strong",{parentName:"p"},"Learning Interpretable Latent Autoencoder Representations with Annotations of Feature Sets")),(0,n.kt)("p",null,"S. Rybakov, ",(0,n.kt)("u",null,"M. Lotfollahi"),", F.J. Theis",(0,n.kt)("em",{parentName:"p"},", F.A. Wolf"),"."),(0,n.kt)("p",null,(0,n.kt)("a",{parentName:"p",href:"https://doi.org/10.1101/2020.12.02.401182"},"[Machine Learning in Computational Biology (2020)]"),",\n",(0,n.kt)("a",{parentName:"p",href:"https://github.com/theislab/intercode"},"[code]"),"."),(0,n.kt)("h3",{id:"2019"},"2019"),(0,n.kt)("p",null,(0,n.kt)("strong",{parentName:"p"},"scGen predicts single-cell perturbation responses.")),(0,n.kt)("u",null,"Lotfollahi, M."),", Wolf, F. A. & Theis, F. J.",(0,n.kt)("p",null,(0,n.kt)("a",{parentName:"p",href:"https://doi.org/10.1038/s41592-019-0494-8"},"[","Nature Methods(2019)","]"),",\n",(0,n.kt)("a",{parentName:"p",href:"https://github.com/theislab/scgen"},"[","code","]"),",\n",(0,n.kt)("a",{parentName:"p",href:"https://www.bioengineering.tum.de/en/news/details/ai-extrapolates-from-mice-to-humans-1"},"[","press","]"),"."),(0,n.kt)("hr",null),(0,n.kt)("h5",{id:"deep-packet-a-novel-approach-for-encrypted-traffic-classification-using-deep-learning"},"Deep packet: a novel approach for encrypted traffic classification using deep learning"),(0,n.kt)("u",null,"Lotfollahi, M."),", Jafari Siavoshani, M., Shirali Hossein Zade, R., & Saberian, M.",(0,n.kt)("p",null,(0,n.kt)("a",{parentName:"p",href:"https://link.springer.com/article/10.1007/s00500-019-04030-2?wt_mc=Internal.Event.1.SEM.ArticleAuthorOnlineFirst&utm_source=ArticleAuthorOnlineFirst&utm_medium=email&utm_content=AA_en_06082018&ArticleAuthorOnlineFirst_20190516&error=cookies_not_supported&code=04315f4e-0ae2-48fb-9e2f-c2d25db1a332"},"[","soft computing(2019)","]"),"."))}u.isMDXComponent=!0}}]);