"use strict";(globalThis.webpackChunklotfollahi_com=globalThis.webpackChunklotfollahi_com||[]).push([[6085],{4547:(e,a,s)=>{s.r(a),s.d(a,{contentTitle:()=>o,default:()=>b,frontMatter:()=>i,metadata:()=>r,toc:()=>n});var t=s(8168),l=(s(6540),s(5680));const i={title:"Publications",hide_table_of_contents:!1},o=void 0,r={type:"mdx",permalink:"/publications",source:"@site/src/pages/publications.md",title:"Publications",description:"- See a full list of papers on Google Scholar.",frontMatter:{title:"Publications",hide_table_of_contents:!1}},n=[],g=(c="Layout",function(e){return console.warn("Component "+c+" was not imported, exported, or provided by MDXProvider as global scope"),(0,l.yg)("div",e)});var c;const u={toc:n},p="wrapper";function b({components:e,...a}){return(0,l.yg)(p,(0,t.A)({},u,a,{components:e,mdxType:"MDXLayout"}),(0,l.yg)(g,{mdxType:"Layout"},(0,l.yg)("div",{className:"pub-wrap"},(0,l.yg)("ul",null,(0,l.yg)("li",{parentName:"ul"},"See a full list of papers on ",(0,l.yg)("a",{parentName:"li",href:"https://scholar.google.com/citations?hl=en&user=NXhouUcAAAAJ&view_op=list_works&sortby=pubdate"},(0,l.yg)("strong",{parentName:"a"},"Google Scholar")),"."),(0,l.yg)("li",{parentName:"ul"},"Co-first and co-last authors are indicated by + and ","*",", respectively.")),(0,l.yg)("h3",{className:"pub-year"},"2025"),(0,l.yg)("div",{className:"pub-list"},(0,l.yg)("div",{className:"pub-item"},(0,l.yg)("div",{className:"pub-title"},"Quantitative characterization of cell niches in spatially resolved omics data."),(0,l.yg)("div",{className:"pub-meta"},"Birk, S., Bonafonte-Pard\xe0s, I., Feriz, A. M., Boxall, A., Agirre, E., Memi, F., ... & ",(0,l.yg)("strong",null,"Lotfollahi, M.")),(0,l.yg)("div",{className:"pub-badges"},(0,l.yg)("a",{href:"https://www.nature.com/articles/s41588-024-01968-3",className:"pub-badge pub-badge--journal"},"Nature Genetics (2025)"),(0,l.yg)("a",{href:"https://github.com/Lotfollahi-lab/cnmf",className:"pub-badge pub-badge--code"},"code"))),(0,l.yg)("div",{className:"pub-item"},(0,l.yg)("div",{className:"pub-title"},"Predicting cell morphological responses to perturbations using generative modeling."),(0,l.yg)("div",{className:"pub-meta"},"Palma, A., Theis, F. J., & ",(0,l.yg)("strong",null,"Lotfollahi, M.")),(0,l.yg)("div",{className:"pub-badges"},(0,l.yg)("a",{href:"https://www.nature.com/articles/s41467-024-57777-w",className:"pub-badge pub-badge--journal"},"Nature Communications (2025)"),(0,l.yg)("a",{href:"https://github.com/theislab/morpho-generation",className:"pub-badge pub-badge--code"},"code"))),(0,l.yg)("div",{className:"pub-item"},(0,l.yg)("div",{className:"pub-title"},"Towards multimodal foundation models in molecular cell biology."),(0,l.yg)("div",{className:"pub-meta"},"Cui, H., Tejada-Lapuerta, A., Brbi\u0107, M., Saez-Rodriguez, J., Cristea, S., ... & ",(0,l.yg)("strong",null,"Lotfollahi, M."),", ... & Wang, B."),(0,l.yg)("div",{className:"pub-badges"},(0,l.yg)("a",{href:"https://www.nature.com/articles/s41586-025-07611-x",className:"pub-badge pub-badge--journal"},"Nature (2025)"))),(0,l.yg)("div",{className:"pub-item"},(0,l.yg)("div",{className:"pub-title"},"Integration and querying of multimodal single-cell data with PoE-VAE."),(0,l.yg)("div",{className:"pub-meta"},"Litinetskaya, A., Schulman, M., Curion, F., Szalata, A., Omidi, A., ",(0,l.yg)("strong",null,"Lotfollahi, M."),", & Theis, F. J."),(0,l.yg)("div",{className:"pub-badges"},(0,l.yg)("a",{href:"https://dl.acm.org/doi/abs/10.1145/3694348.3713023",className:"pub-badge pub-badge--preprint"},"RECOMB (2025)"))),(0,l.yg)("div",{className:"pub-item"},(0,l.yg)("div",{className:"pub-title"},"Integrating multi-covariate disentanglement with counterfactual analysis on synthetic data enables cell type discovery and counterfactual predictions."),(0,l.yg)("div",{className:"pub-meta"},"Megas, S., Amani, A., Rose, A., Dufva, O., Shamsaie, K., Asadollahzadeh, H., ... & ",(0,l.yg)("strong",null,"Lotfollahi, M.")),(0,l.yg)("div",{className:"pub-badges"},(0,l.yg)("a",{href:"https://www.biorxiv.org/content/10.1101/2025.06.03.657578v1",className:"pub-badge pub-badge--preprint"},"bioRxiv (2025)"))),(0,l.yg)("div",{className:"pub-item"},(0,l.yg)("div",{className:"pub-title"},"Mapping and reprogramming human tissue microenvironments with MintFlow."),(0,l.yg)("div",{className:"pub-meta"},"Akbarnejad, A., Steele, L., Jafree, D. J., Birk, S., Sallese, M. R., Rademaker, K., ... & ",(0,l.yg)("strong",null,"Lotfollahi, M.")),(0,l.yg)("div",{className:"pub-badges"},(0,l.yg)("a",{href:"https://www.biorxiv.org/content/10.1101/2025.06.24.661094v1",className:"pub-badge pub-badge--preprint"},"bioRxiv (2025)")))),(0,l.yg)("h3",{className:"pub-year"},"2024"),(0,l.yg)("div",{className:"pub-list"},(0,l.yg)("div",{className:"pub-item"},(0,l.yg)("div",{className:"pub-title"},"Multi-modal generative modeling for joint analysis of single-cell T cell receptor and gene expression data."),(0,l.yg)("div",{className:"pub-meta"},"Drost, F., An, Y., Bonafonte-Pard\xe0s, I., Dratva, L. M., Lindeboom, R. G., Haniffa, M., ... , ",(0,l.yg)("strong",null,"Lotfollahi, M.*"),", & Schubert, B.*"),(0,l.yg)("div",{className:"pub-badges"},(0,l.yg)("a",{href:"https://www.nature.com/articles/s41467-024-49774-6",className:"pub-badge pub-badge--journal"},"Nature Communications (2024)"),(0,l.yg)("a",{href:"https://github.com/theislab/scar",className:"pub-badge pub-badge--code"},"code"))),(0,l.yg)("div",{className:"pub-item"},(0,l.yg)("div",{className:"pub-title"},"A single cell and spatial genomics atlas of human skin fibroblasts in health and disease."),(0,l.yg)("div",{className:"pub-meta"},"Steele, L., Admane, C., Chakala, K. P., Foster, A., Gopee, N. H., Koplev, S., ... & ",(0,l.yg)("strong",null,"Lotfollahi, M.*"),", & Haniffa, M.*"),(0,l.yg)("div",{className:"pub-badges"},(0,l.yg)("a",{href:"https://www.biorxiv.org/content/10.1101/2024.12.23.629194v1",className:"pub-badge pub-badge--preprint"},"bioRxiv (2024)"))),(0,l.yg)("div",{className:"pub-item"},(0,l.yg)("div",{className:"pub-title"},"ArchMap: A web-based platform for reference-based analysis of single-cell datasets."),(0,l.yg)("div",{className:"pub-meta"},(0,l.yg)("strong",null,"Lotfollahi, M."),", Bright, C., Skorobogat, R., Dehkordi, M., George, X., Richter, S., ... & Theis, F. J."),(0,l.yg)("div",{className:"pub-badges"},(0,l.yg)("a",{href:"https://www.biorxiv.org/content/10.1101/2024.09.19.613883v1",className:"pub-badge pub-badge--preprint"},"bioRxiv (2024)"))),(0,l.yg)("div",{className:"pub-item"},(0,l.yg)("div",{className:"pub-title"},"Pertpy: an end-to-end framework for perturbation analysis."),(0,l.yg)("div",{className:"pub-meta"},"Heumos, L., Ji, Y., May, L., Green, T., Zhang, X., Wu, X., Ostner, J., Peidli, S., ... & ",(0,l.yg)("strong",null,"Lotfollahi, M.*"),", & Theis, F. J.*"),(0,l.yg)("div",{className:"pub-badges"},(0,l.yg)("a",{href:"https://www.biorxiv.org/content/10.1101/2024.08.04.606516v1",className:"pub-badge pub-badge--preprint"},"bioRxiv (2024)"),(0,l.yg)("a",{href:"https://github.com/scverse/pertpy",className:"pub-badge pub-badge--code"},"code"))),(0,l.yg)("div",{className:"pub-item"},(0,l.yg)("div",{className:"pub-title"},"Toward learning a foundational representation of cells and genes."),(0,l.yg)("div",{className:"pub-meta"},(0,l.yg)("strong",null,"Lotfollahi, M.")),(0,l.yg)("div",{className:"pub-badges"},(0,l.yg)("a",{href:"https://www.nature.com/articles/s41592-024-02298-6",className:"pub-badge pub-badge--journal"},"Nature Methods (2024)"))),(0,l.yg)("div",{className:"pub-item"},(0,l.yg)("div",{className:"pub-title"},"The future of rapid and automated single-cell data analysis using reference mapping."),(0,l.yg)("div",{className:"pub-meta"},(0,l.yg)("strong",null,"Lotfollahi, M."),", Hao, Y., Theis, F. J., & Satija, R."),(0,l.yg)("div",{className:"pub-badges"},(0,l.yg)("a",{href:"https://www.cell.com/cell/fulltext/S0092-8674(24)00405-2",className:"pub-badge pub-badge--journal"},"Cell (2024)"))),(0,l.yg)("div",{className:"pub-item"},(0,l.yg)("div",{className:"pub-title"},"Deep learning in spatially resolved transcriptomics: a comprehensive technical view."),(0,l.yg)("div",{className:"pub-meta"},"Zahedi, R., Ghamsari, R., Argha, A., Macphillamy, C., Beheshti, A., ... & ",(0,l.yg)("strong",null,"Lotfollahi, M.*"),", & Alinejad-Rokny, H.*"),(0,l.yg)("div",{className:"pub-badges"},(0,l.yg)("a",{href:"https://academic.oup.com/bib/article/25/2/bbae082/7615995",className:"pub-badge pub-badge--journal"},"Briefings in Bioinformatics (2024)"))),(0,l.yg)("div",{className:"pub-item"},(0,l.yg)("div",{className:"pub-title"},"Multimodal weakly supervised learning to identify disease-specific changes in single-cell atlases."),(0,l.yg)("div",{className:"pub-meta"},"Litinetskaya, A., Shulman, M., Hediyeh-zadeh, S., Moinfar, A. A., Curion, F., ... & ",(0,l.yg)("strong",null,"Lotfollahi, M.*"),", & Theis, F. J.*"),(0,l.yg)("div",{className:"pub-badges"},(0,l.yg)("a",{href:"https://www.biorxiv.org/content/10.1101/2024.07.29.605625v1",className:"pub-badge pub-badge--preprint"},"bioRxiv (2024)"))),(0,l.yg)("div",{className:"pub-item"},(0,l.yg)("div",{className:"pub-title"},"Deep generative modeling of transcriptional dynamics for RNA velocity analysis in single cells."),(0,l.yg)("div",{className:"pub-meta"},"Gayoso, A., Weiler, P., ",(0,l.yg)("strong",null,"Lotfollahi, M."),", Klein, D., Hong, J., Streets, A., Theis, F. J., & Yosef, N."),(0,l.yg)("div",{className:"pub-badges"},(0,l.yg)("a",{href:"https://www.nature.com/articles/s41592-023-02131-8",className:"pub-badge pub-badge--journal"},"Nature Methods (2024)"))),(0,l.yg)("div",{className:"pub-item"},(0,l.yg)("div",{className:"pub-title"},"An integrated single-cell reference atlas of the human endometrium."),(0,l.yg)("div",{className:"pub-meta"},"Mare\u010dkov\xe1, M., Garcia-Alonso, L., Moullet, M., Lorenzi, V., Petryszak, R., ... & ",(0,l.yg)("strong",null,"Lotfollahi, M."),", ... & Vento-Tormo, R."),(0,l.yg)("div",{className:"pub-badges"},(0,l.yg)("a",{href:"https://www.nature.com/articles/s41588-024-01859-7",className:"pub-badge pub-badge--journal"},"Nature Genetics (2024)")))),(0,l.yg)("h3",{className:"pub-year"},"2023"),(0,l.yg)("div",{className:"pub-list"},(0,l.yg)("div",{className:"pub-item"},(0,l.yg)("div",{className:"pub-title"},"Population-level integration of single-cell datasets enables multi-scale analysis across samples."),(0,l.yg)("div",{className:"pub-meta"},"De Donno, C., Hediyeh-Zadeh, S., Moinfar, A. A., Wagenstetter, M., Zappia, L., ... & ",(0,l.yg)("strong",null,"Lotfollahi, M."),", & Theis, F. J."),(0,l.yg)("div",{className:"pub-badges"},(0,l.yg)("a",{href:"https://www.nature.com/articles/s41592-023-02056-2",className:"pub-badge pub-badge--journal"},"Nature Methods (2023)"),(0,l.yg)("a",{href:"https://github.com/theislab/scarches-reproducibility/tree/master/population_extension",className:"pub-badge pub-badge--code"},"code"))),(0,l.yg)("div",{className:"pub-item"},(0,l.yg)("div",{className:"pub-title"},"Single-cell reference mapping to construct and extend cell-type hierarchies."),(0,l.yg)("div",{className:"pub-meta"},"Michielsen, L.+, ",(0,l.yg)("strong",null,"Lotfollahi, M."),"+, Strobl, D., Sikkema, L., Reinders, M. J. T., Theis, F. J., & Mahfouz, A."),(0,l.yg)("div",{className:"pub-badges"},(0,l.yg)("a",{href:"https://academic.oup.com/nargab/article/5/3/lqad070/7253509",className:"pub-badge pub-badge--journal"},"NAR Genomics and Bioinformatics (2023)"),(0,l.yg)("a",{href:"https://github.com/LMichielsen/scHPL",className:"pub-badge pub-badge--code"},"code"))),(0,l.yg)("div",{className:"pub-item"},(0,l.yg)("div",{className:"pub-title"},"Best practices for single-cell analysis across modalities."),(0,l.yg)("div",{className:"pub-meta"},"Heumos, L., Schaar, A. C., Lance, C., Litinetskaya, A., Drost, F., Zappia, L., ... & ",(0,l.yg)("strong",null,"Lotfollahi, M."),", ... & Theis, F. J."),(0,l.yg)("div",{className:"pub-badges"},(0,l.yg)("a",{href:"https://www.nature.com/articles/s41576-023-00586-w",className:"pub-badge pub-badge--journal"},"Nature Reviews Genetics (2023)"))),(0,l.yg)("div",{className:"pub-item"},(0,l.yg)("div",{className:"pub-title"},"An integrated cell atlas of the human lung in health and disease."),(0,l.yg)("div",{className:"pub-meta"},"Sikkema, L., Ram\xedrez-Su\xe1stegui, C., Strobl, D. C., Gillett, T. E., Zappia, L., ... & ",(0,l.yg)("strong",null,"Lotfollahi, M."),", ... & Theis, F. J."),(0,l.yg)("div",{className:"pub-badges"},(0,l.yg)("a",{href:"https://www.nature.com/articles/s41591-023-02327-2",className:"pub-badge pub-badge--journal"},"Nature Medicine (2023)"))),(0,l.yg)("div",{className:"pub-item"},(0,l.yg)("div",{className:"pub-title"},"Predicting cellular responses to complex perturbations in high\u2010throughput screens."),(0,l.yg)("div",{className:"pub-meta"},(0,l.yg)("strong",null,"Lotfollahi, M."),"+, Susmelj, A. K.+, De Donno, C.+, Hetzel, L., Ji, Y., Ibarra, I. L., ... & Theis, F. J."),(0,l.yg)("div",{className:"pub-badges"},(0,l.yg)("a",{href:"https://www.embopress.org/doi/full/10.15252/msb.202211517",className:"pub-badge pub-badge--journal"},"Molecular Systems Biology (2023)"),(0,l.yg)("a",{href:"https://github.com/theislab/cpa-reproducibility",className:"pub-badge pub-badge--code"},"code"))),(0,l.yg)("div",{className:"pub-item"},(0,l.yg)("div",{className:"pub-title"},"The scverse project provides a computational ecosystem for single-cell omics data analysis."),(0,l.yg)("div",{className:"pub-meta"},"Virshup, I., Bredikhin, D., Heumos, L., Palla, G., Sturm, G., Gayoso, A., ... & ",(0,l.yg)("strong",null,"Lotfollahi, M."),", ... & Theis, F. J."),(0,l.yg)("div",{className:"pub-badges"},(0,l.yg)("a",{href:"https://www.nature.com/articles/s41587-023-01733-8",className:"pub-badge pub-badge--journal"},"Nature Biotechnology (2023)"))),(0,l.yg)("div",{className:"pub-item"},(0,l.yg)("div",{className:"pub-title"},"Mapping cells to gene programs."),(0,l.yg)("div",{className:"pub-meta"},(0,l.yg)("strong",null,"Lotfollahi, M."),", Rybakov, S., Hrovatin, K., Hediyeh-zadeh, S., Talavera-Lopez, C., Misharin, A. V., & Theis, F. J."),(0,l.yg)("div",{className:"pub-badges"},(0,l.yg)("a",{href:"https://protocolexchange.researchsquare.com/article/pex-2066/v1",className:"pub-badge pub-badge--preprint"},"Protocol Exchange (2023)"))),(0,l.yg)("div",{className:"pub-item"},(0,l.yg)("div",{className:"pub-title"},"Biologically informed deep learning to query gene programs in single-cell atlases."),(0,l.yg)("div",{className:"pub-meta"},(0,l.yg)("strong",null,"Lotfollahi, M."),"+, Rybakov, S.+, Hrovatin, K., Hediyeh-Zadeh, S., Talavera-L\xf3pez, C., ... & Misharin, A. V., & Theis, F. J."),(0,l.yg)("div",{className:"pub-badges"},(0,l.yg)("a",{href:"https://www.nature.com/articles/s41556-022-01072-x",className:"pub-badge pub-badge--journal"},"Nature Cell Biology (2023)"),(0,l.yg)("a",{href:"https://github.com/theislab/expiby-reproducibility",className:"pub-badge pub-badge--code"},"code")))),(0,l.yg)("h3",{className:"pub-year"},"2022"),(0,l.yg)("div",{className:"pub-list"},(0,l.yg)("div",{className:"pub-item"},(0,l.yg)("div",{className:"pub-title"},"Modelling method using a conditional variational autoencoder."),(0,l.yg)("div",{className:"pub-meta"},"Theis, F. J., ",(0,l.yg)("strong",null,"Lotfollahi, M."),", & Wolf, F. A."),(0,l.yg)("div",{className:"pub-badges"},(0,l.yg)("a",{href:"https://patents.google.com/patent/US20220237976A1/en",className:"pub-badge"},"US Patent (2022)"))),(0,l.yg)("div",{className:"pub-item"},(0,l.yg)("div",{className:"pub-title"},"Squidpy: a scalable framework for spatial omics analysis."),(0,l.yg)("div",{className:"pub-meta"},"Palla, G., Spitzer, H., Klein, M., Fischer, D., Schaar, A. C., Kuemmerle, L. B., ... & ",(0,l.yg)("strong",null,"Lotfollahi, M."),", ... & Theis, F. J."),(0,l.yg)("div",{className:"pub-badges"},(0,l.yg)("a",{href:"https://www.nature.com/articles/s41592-021-01358-2",className:"pub-badge pub-badge--journal"},"Nature Methods (2022)"),(0,l.yg)("a",{href:"https://github.com/scverse/squidpy",className:"pub-badge pub-badge--code"},"code"))),(0,l.yg)("div",{className:"pub-item"},(0,l.yg)("div",{className:"pub-title"},"A Python library for probabilistic analysis of single-cell omics data."),(0,l.yg)("div",{className:"pub-meta"},"Gayoso, A.+, Lopez, R.+, Xing, G., Boyeau, P., ... & ",(0,l.yg)("strong",null,"Lotfollahi, M."),", \u2026 & Yosef, N."),(0,l.yg)("div",{className:"pub-badges"},(0,l.yg)("a",{href:"https://www.nature.com/articles/s41587-021-01000-7",className:"pub-badge pub-badge--journal"},"Nature Biotechnology (2022)"),(0,l.yg)("a",{href:"https://github.com/scverse/scvi-tools",className:"pub-badge pub-badge--code"},"code"))),(0,l.yg)("div",{className:"pub-item"},(0,l.yg)("div",{className:"pub-title"},"Mapping single-cell data to reference atlases by transfer learning."),(0,l.yg)("div",{className:"pub-meta"},(0,l.yg)("strong",null,"Lotfollahi, M."),", Naghipourfar, M., Luecken, M. D., Khajavi, M., B\xfcttner, M., ... & Theis, F. J."),(0,l.yg)("div",{className:"pub-badges"},(0,l.yg)("a",{href:"https://www.nature.com/articles/s41587-021-01001-6",className:"pub-badge pub-badge--journal"},"Nature Biotechnology (2022)"),(0,l.yg)("a",{href:"https://github.com/theislab/scarches",className:"pub-badge pub-badge--code"},"code")))),(0,l.yg)("h3",{className:"pub-year"},"2021"),(0,l.yg)("div",{className:"pub-list"},(0,l.yg)("div",{className:"pub-item"},(0,l.yg)("div",{className:"pub-title"},"Machine learning for perturbational single-cell omics."),(0,l.yg)("div",{className:"pub-meta"},"Ji, Y., ",(0,l.yg)("strong",null,"Lotfollahi, M."),", Wolf, F. A., & Theis, F. J."),(0,l.yg)("div",{className:"pub-badges"},(0,l.yg)("a",{href:"https://www.cell.com/cell-systems/fulltext/S2405-4712(21)00155-2",className:"pub-badge pub-badge--journal"},"Cell Systems (2021)")))),(0,l.yg)("h3",{className:"pub-year"},"2020"),(0,l.yg)("div",{className:"pub-list"},(0,l.yg)("div",{className:"pub-item"},(0,l.yg)("div",{className:"pub-title"},"Conditional out-of-distribution generation for unpaired data using transfer VAE."),(0,l.yg)("div",{className:"pub-meta"},(0,l.yg)("strong",null,"Lotfollahi, M."),", Naghipourfar, M., Theis, F. J., & Wolf, F. A."),(0,l.yg)("div",{className:"pub-badges"},(0,l.yg)("a",{href:"https://academic.oup.com/bioinformatics/article/36/Supplement_2/i610/6055866",className:"pub-badge pub-badge--journal"},"Bioinformatics (2020)")))),(0,l.yg)("h3",{className:"pub-year"},"2019"),(0,l.yg)("div",{className:"pub-list"},(0,l.yg)("div",{className:"pub-item"},(0,l.yg)("div",{className:"pub-title"},"scGen predicts single-cell perturbation responses."),(0,l.yg)("div",{className:"pub-meta"},(0,l.yg)("strong",null,"Lotfollahi, M."),", Wolf, F. A., & Theis, F. J."),(0,l.yg)("div",{className:"pub-badges"},(0,l.yg)("a",{href:"https://www.nature.com/articles/s41592-019-0494-8",className:"pub-badge pub-badge--journal"},"Nature Methods (2019)"),(0,l.yg)("a",{href:"https://github.com/theislab/scgen",className:"pub-badge pub-badge--code"},"code"))),(0,l.yg)("div",{className:"pub-item"},(0,l.yg)("div",{className:"pub-title"},"Deep packet: A novel approach for encrypted traffic classification using deep learning."),(0,l.yg)("div",{className:"pub-meta"},(0,l.yg)("strong",null,"Lotfollahi, M."),", Siavoshani, M. J., Zade, R. S. H., & Saberian, M."),(0,l.yg)("div",{className:"pub-badges"},(0,l.yg)("a",{href:"https://link.springer.com/article/10.1007/s00500-019-04033-5",className:"pub-badge pub-badge--journal"},"Soft Computing (2019)")))))))}b.isMDXComponent=!0},5680:(e,a,s)=>{s.d(a,{xA:()=>c,yg:()=>d});var t=s(6540);function l(e,a,s){return a in e?Object.defineProperty(e,a,{value:s,enumerable:!0,configurable:!0,writable:!0}):e[a]=s,e}function i(e,a){var s=Object.keys(e);if(Object.getOwnPropertySymbols){var t=Object.getOwnPropertySymbols(e);a&&(t=t.filter(function(a){return Object.getOwnPropertyDescriptor(e,a).enumerable})),s.push.apply(s,t)}return s}function o(e){for(var a=1;a<arguments.length;a++){var s=null!=arguments[a]?arguments[a]:{};a%2?i(Object(s),!0).forEach(function(a){l(e,a,s[a])}):Object.getOwnPropertyDescriptors?Object.defineProperties(e,Object.getOwnPropertyDescriptors(s)):i(Object(s)).forEach(function(a){Object.defineProperty(e,a,Object.getOwnPropertyDescriptor(s,a))})}return e}function r(e,a){if(null==e)return{};var s,t,l=function(e,a){if(null==e)return{};var s,t,l={},i=Object.keys(e);for(t=0;t<i.length;t++)s=i[t],a.indexOf(s)>=0||(l[s]=e[s]);return l}(e,a);if(Object.getOwnPropertySymbols){var i=Object.getOwnPropertySymbols(e);for(t=0;t<i.length;t++)s=i[t],a.indexOf(s)>=0||Object.prototype.propertyIsEnumerable.call(e,s)&&(l[s]=e[s])}return l}var n=t.createContext({}),g=function(e){var a=t.useContext(n),s=a;return e&&(s="function"==typeof e?e(a):o(o({},a),e)),s},c=function(e){var a=g(e.components);return t.createElement(n.Provider,{value:a},e.children)},u="mdxType",p={inlineCode:"code",wrapper:function(e){var a=e.children;return t.createElement(t.Fragment,{},a)}},b=t.forwardRef(function(e,a){var s=e.components,l=e.mdxType,i=e.originalType,n=e.parentName,c=r(e,["components","mdxType","originalType","parentName"]),u=g(s),b=l,d=u["".concat(n,".").concat(b)]||u[b]||p[b]||i;return s?t.createElement(d,o(o({ref:a},c),{},{components:s})):t.createElement(d,o({ref:a},c))});function d(e,a){var s=arguments,l=a&&a.mdxType;if("string"==typeof e||l){var i=s.length,o=new Array(i);o[0]=b;var r={};for(var n in a)hasOwnProperty.call(a,n)&&(r[n]=a[n]);r.originalType=e,r[u]="string"==typeof e?e:l,o[1]=r;for(var g=2;g<i;g++)o[g]=s[g];return t.createElement.apply(null,o)}return t.createElement.apply(null,s)}b.displayName="MDXCreateElement"}}]);