<!doctype html>
<html lang="en" dir="ltr" class="mdx-wrapper mdx-page plugin-pages plugin-id-default">
<head>
<meta charset="UTF-8">
<meta name="generator" content="Docusaurus v2.4.1">
<title data-rh="true"> Lotfollahi</title><meta data-rh="true" property="og:title" content=" Lotfollahi"><meta data-rh="true" name="viewport" content="width=device-width,initial-scale=1"><meta data-rh="true" name="twitter:card" content="summary_large_image"><meta data-rh="true" property="og:url" content="https://demo.lotfollahi.com/publications"><meta data-rh="true" name="docusaurus_locale" content="en"><meta data-rh="true" name="docusaurus_tag" content="default"><meta data-rh="true" name="docsearch:language" content="en"><meta data-rh="true" name="docsearch:docusaurus_tag" content="default"><meta data-rh="true" name="description" content="- See a full list of papers on Google Scholar."><meta data-rh="true" property="og:description" content="- See a full list of papers on Google Scholar."><link data-rh="true" rel="icon" href="/img/favicon.ico"><link data-rh="true" rel="canonical" href="https://demo.lotfollahi.com/publications"><link data-rh="true" rel="alternate" href="https://demo.lotfollahi.com/publications" hreflang="en"><link data-rh="true" rel="alternate" href="https://demo.lotfollahi.com/publications" hreflang="x-default"><link rel="alternate" type="application/rss+xml" href="/blog/rss.xml" title=" Lotfollahi RSS Feed">
<link rel="alternate" type="application/atom+xml" href="/blog/atom.xml" title=" Lotfollahi Atom Feed"><link rel="stylesheet" href="/assets/css/styles.bb8174c5.css">
<link rel="preload" href="/assets/js/runtime~main.49356ac3.js" as="script">
<link rel="preload" href="/assets/js/main.52fb0cbd.js" as="script">
</head>
<body class="navigation-with-keyboard">
<script>!function(){function t(t){document.documentElement.setAttribute("data-theme",t)}var e=function(){var t=null;try{t=new URLSearchParams(window.location.search).get("docusaurus-theme")}catch(t){}return t}()||function(){var t=null;try{t=localStorage.getItem("theme")}catch(t){}return t}();t(null!==e?e:"light")}()</script><div id="__docusaurus">
<div role="region" aria-label="Skip to main content"><a class="skipToContent_fXgn" href="#__docusaurus_skipToContent_fallback">Skip to main content</a></div><nav aria-label="Main" class="navbar navbar--fixed-top"><div class="navbar__inner"><div class="navbar__items"><button aria-label="Toggle navigation bar" aria-expanded="false" class="navbar__toggle clean-btn" type="button"><svg width="30" height="30" viewBox="0 0 30 30" aria-hidden="true"><path stroke="currentColor" stroke-linecap="round" stroke-miterlimit="10" stroke-width="2" d="M4 7h22M4 15h22M4 23h22"></path></svg></button><a class="navbar__brand" href="/"></a><a aria-current="page" class="navbar__item navbar__link navbar__link--active" href="/">Home</a><a class="navbar__item navbar__link" href="/about">About</a><a class="navbar__item navbar__link" href="/Lab"> The Lab</a><a class="navbar__item navbar__link" href="/research">Research</a><a aria-current="page" class="navbar__item navbar__link navbar__link--active" href="/publications">Publications</a><a class="navbar__item navbar__link" href="/blog">Blog/news</a><a class="navbar__item navbar__link" href="/software">Software</a><a class="navbar__item navbar__link" href="/talks">Talks</a></div><div class="navbar__items navbar__items--right"><div class="toggle_vylO colorModeToggle_DEke"><button class="clean-btn toggleButton_gllP toggleButtonDisabled_aARS" type="button" disabled="" title="Switch between dark and light mode (currently light mode)" aria-label="Switch between dark and light mode (currently light mode)" aria-live="polite"><svg viewBox="0 0 24 24" width="24" height="24" class="lightToggleIcon_pyhR"><path fill="currentColor" d="M12,9c1.65,0,3,1.35,3,3s-1.35,3-3,3s-3-1.35-3-3S10.35,9,12,9 M12,7c-2.76,0-5,2.24-5,5s2.24,5,5,5s5-2.24,5-5 S14.76,7,12,7L12,7z M2,13l2,0c0.55,0,1-0.45,1-1s-0.45-1-1-1l-2,0c-0.55,0-1,0.45-1,1S1.45,13,2,13z M20,13l2,0c0.55,0,1-0.45,1-1 s-0.45-1-1-1l-2,0c-0.55,0-1,0.45-1,1S19.45,13,20,13z M11,2v2c0,0.55,0.45,1,1,1s1-0.45,1-1V2c0-0.55-0.45-1-1-1S11,1.45,11,2z M11,20v2c0,0.55,0.45,1,1,1s1-0.45,1-1v-2c0-0.55-0.45-1-1-1C11.45,19,11,19.45,11,20z M5.99,4.58c-0.39-0.39-1.03-0.39-1.41,0 c-0.39,0.39-0.39,1.03,0,1.41l1.06,1.06c0.39,0.39,1.03,0.39,1.41,0s0.39-1.03,0-1.41L5.99,4.58z M18.36,16.95 c-0.39-0.39-1.03-0.39-1.41,0c-0.39,0.39-0.39,1.03,0,1.41l1.06,1.06c0.39,0.39,1.03,0.39,1.41,0c0.39-0.39,0.39-1.03,0-1.41 L18.36,16.95z M19.42,5.99c0.39-0.39,0.39-1.03,0-1.41c-0.39-0.39-1.03-0.39-1.41,0l-1.06,1.06c-0.39,0.39-0.39,1.03,0,1.41 s1.03,0.39,1.41,0L19.42,5.99z M7.05,18.36c0.39-0.39,0.39-1.03,0-1.41c-0.39-0.39-1.03-0.39-1.41,0l-1.06,1.06 c-0.39,0.39-0.39,1.03,0,1.41s1.03,0.39,1.41,0L7.05,18.36z"></path></svg><svg viewBox="0 0 24 24" width="24" height="24" class="darkToggleIcon_wfgR"><path fill="currentColor" d="M9.37,5.51C9.19,6.15,9.1,6.82,9.1,7.5c0,4.08,3.32,7.4,7.4,7.4c0.68,0,1.35-0.09,1.99-0.27C17.45,17.19,14.93,19,12,19 c-3.86,0-7-3.14-7-7C5,9.07,6.81,6.55,9.37,5.51z M12,3c-4.97,0-9,4.03-9,9s4.03,9,9,9s9-4.03,9-9c0-0.46-0.04-0.92-0.1-1.36 c-0.98,1.37-2.58,2.26-4.4,2.26c-2.98,0-5.4-2.42-5.4-5.4c0-1.81,0.89-3.42,2.26-4.4C12.92,3.04,12.46,3,12,3L12,3z"></path></svg></button></div><div class="searchBox_ZlJk"></div></div></div><div role="presentation" class="navbar-sidebar__backdrop"></div></nav><div id="__docusaurus_skipToContent_fallback" class="main-wrapper mainWrapper_z2l0"><main class="container container--fluid margin-vert--lg"><div class="row mdxPageWrapper_j9I6"><div class="col col--8"><article><ul><li>See a full list of papers on <strong><a href="https://scholar.google.com/citations?user=NXhouUcAAAAJ&amp;hl=en" target="_blank" rel="noopener noreferrer">Google Scholar</a></strong>.</li><li>Co-first and co-last authors are indicated by + and *, respectively.</li></ul><h3 class="anchor anchorWithStickyNavbar_LWe7" id="2024">2024<a href="#2024" class="hash-link" aria-label="Direct link to 2024" title="Direct link to 2024">​</a></h3><h5 class="anchor anchorWithStickyNavbar_LWe7" id="multi-modal-generative-modeling-for-joint-analysis-of-single-cell-t-cell-receptor-and-gene-expression-data">Multi-modal generative modeling for joint analysis of single-cell T cell receptor and gene expression data.<a href="#multi-modal-generative-modeling-for-joint-analysis-of-single-cell-t-cell-receptor-and-gene-expression-data" class="hash-link" aria-label="Direct link to Multi-modal generative modeling for joint analysis of single-cell T cell receptor and gene expression data." title="Direct link to Multi-modal generative modeling for joint analysis of single-cell T cell receptor and gene expression data.">​</a></h5><p>Drost, F., An, Y., Bonafonte-Pardàs, I., Dratva, L. M., Lindeboom, R. G., Haniffa, M., ... , <u>Lotfollahi,M<!-- -->*</u>, Schubert, B*. </p><p><a href="https://github.com/SchubertLab/mvTCR" target="_blank" rel="noopener noreferrer">[code]</a>,
<a href="https://www.nature.com/articles/s41467-024-49806-9" target="_blank" rel="noopener noreferrer">[Nature Communications (2024)]</a>.</p><h5 class="anchor anchorWithStickyNavbar_LWe7" id="large-scale-characterization-of-cell-niches-in-spatial-atlases-using-bio-inspired-graph-learning">Large-scale characterization of cell niches in spatial atlases using bio-inspired graph learning<a href="#large-scale-characterization-of-cell-niches-in-spatial-atlases-using-bio-inspired-graph-learning" class="hash-link" aria-label="Direct link to Large-scale characterization of cell niches in spatial atlases using bio-inspired graph learning" title="Direct link to Large-scale characterization of cell niches in spatial atlases using bio-inspired graph learning">​</a></h5><p>Birk, S., Bonafonte-Pardàs, I., Feriz, A. M.,, ... &amp; <u>Lotfollahi, M.<!-- -->*</u>.</p><p><a href="https://github.com/lotfollahi-lab" target="_blank" rel="noopener noreferrer">[code]</a>
<a href="https://www.biorxiv.org/content/10.1101/2024.02.21.581428v2" target="_blank" rel="noopener noreferrer">[bioRxv (2024)]</a>.</p><h3 class="anchor anchorWithStickyNavbar_LWe7" id="2023">2023<a href="#2023" class="hash-link" aria-label="Direct link to 2023" title="Direct link to 2023">​</a></h3><h5 class="anchor anchorWithStickyNavbar_LWe7" id="an-integrated-single-cell-reference-atlas-of-the-human-endometrium">An integrated single-cell reference atlas of the human endometrium.<a href="#an-integrated-single-cell-reference-atlas-of-the-human-endometrium" class="hash-link" aria-label="Direct link to An integrated single-cell reference atlas of the human endometrium." title="Direct link to An integrated single-cell reference atlas of the human endometrium.">​</a></h5><p>Marečková, M., Garcia-Alonso, L., Moullet, M., Lorenzi, V., Petryszak, R., Sancho-Serra, C., <u>Lotfollahi,M<!-- -->*</u> ... &amp;
Vento-Tormo, R. (2023).</p><p><a href="https://www.biorxiv.org/content/10.1101/2023.11.03.564728v1.abstract" target="_blank" rel="noopener noreferrer">[bioRxiv (2023)]</a>.</p><h5 class="anchor anchorWithStickyNavbar_LWe7" id="population-level-integration-of-single-cell-datasets-enables-multi-scale-analysis-across-samples">Population-level integration of single-cell datasets enables multi-scale analysis across samples.<a href="#population-level-integration-of-single-cell-datasets-enables-multi-scale-analysis-across-samples" class="hash-link" aria-label="Direct link to Population-level integration of single-cell datasets enables multi-scale analysis across samples." title="Direct link to Population-level integration of single-cell datasets enables multi-scale analysis across samples.">​</a></h5><p>De Donno, C., Hediyeh-Zadeh, S., Wagenstetter, M., Moinfar, A. A., Zappia, L., </p><u>Lotfollahi, M.* </u>, &amp; Theis, F. J*.<p><a href="https://github.com/theislab/scArches" target="_blank" rel="noopener noreferrer">[code]</a>,
<a href="https://www.nature.com/articles/s41592-023-02035-2" target="_blank" rel="noopener noreferrer">[Nature Methods (2023)]</a>.</p><hr><h5 class="anchor anchorWithStickyNavbar_LWe7" id="predicting-cell-morphological-responses-to-perturbations-using-generative-modeling">Predicting cell morphological responses to perturbations using generative modeling<a href="#predicting-cell-morphological-responses-to-perturbations-using-generative-modeling" class="hash-link" aria-label="Direct link to Predicting cell morphological responses to perturbations using generative modeling" title="Direct link to Predicting cell morphological responses to perturbations using generative modeling">​</a></h5><p>Palma, A., Theis, F. J.<!-- -->*<!-- -->, <u>Lotfollahi,M<!-- -->*</u>.</p><p><a href="https://www.biorxiv.org/content/10.1101/2023.07.17.549216v1" target="_blank" rel="noopener noreferrer">[bioRxiv (2023)]</a>.
<a href="https://github.com/theislab/impa" target="_blank" rel="noopener noreferrer">[code]</a>.</p><hr><h5 class="anchor anchorWithStickyNavbar_LWe7" id="an-integrated-cell-atlas-of-the-human-lung-in-health-and-disease">An integrated cell atlas of the human lung in health and disease<a href="#an-integrated-cell-atlas-of-the-human-lung-in-health-and-disease" class="hash-link" aria-label="Direct link to An integrated cell atlas of the human lung in health and disease" title="Direct link to An integrated cell atlas of the human lung in health and disease">​</a></h5><p>Sikkema, L., Strobl, D., Zappia, L., Madissoon, E., Markov, N. S., Zaragosi, L., Ansari, M., Arguel, M., Apperloo, L., Bécavin, C., Berg, M., Chichelnitskiy, E., Chung, M.,
Collin, A., Gay, A. C. A., Hooshiar Kashani, B., Jain, M., Kapellos, T., Kole, T. M., …,<u>Lotfollahi. M</u>,...,Theis, F.</p><p><a href="https://www.nature.com/articles/s41591-023-02327-2" target="_blank" rel="noopener noreferrer">[Nature Medicine (2023)]</a>.
<a href="https://github.com/LungCellAtlas/HLCA_reproducibility" target="_blank" rel="noopener noreferrer">[code]</a>,
<a href="https://docs.scarches.org/en/latest/hlca_map_classify.html" target="_blank" rel="noopener noreferrer">[Mapping data to HLCA using scArhes]</a>,</p><hr><h5 class="anchor anchorWithStickyNavbar_LWe7" id="predicting-cellular-responses-to-complex-perturbations-in-highthroughput-screens">Predicting cellular responses to complex perturbations in high‐throughput screens<a href="#predicting-cellular-responses-to-complex-perturbations-in-highthroughput-screens" class="hash-link" aria-label="Direct link to Predicting cellular responses to complex perturbations in high‐throughput screens" title="Direct link to Predicting cellular responses to complex perturbations in high‐throughput screens">​</a></h5><u>Lotfollahi, M+.</u>, Klimovskaia Susmelj+, A., De Donno, C+., Hetzel, L., Ji, Y., Ibarra, I. L., ... &amp; Theis, F. J.<p><a href="https://www.embopress.org/doi/full/10.15252/msb.202211517" target="_blank" rel="noopener noreferrer">[Molecular Systems Biology (2023)]</a>,
<a href="https://github.com/facebookresearch/CPA" target="_blank" rel="noopener noreferrer">[code]</a>,
<a href="https://ai.facebook.com/blog/ai-predicts-effective-drug-combinations-to-fight-complex-diseases-faster" target="_blank" rel="noopener noreferrer">[Facebook AI blogpost]</a>,
<a href="https://www.stateof.ai/2021" target="_blank" rel="noopener noreferrer">[state of AI report 2021]</a>,
<a href="https://www.embopress.org/loi/17444292" target="_blank" rel="noopener noreferrer">[featured cover]</a>.</p><hr><h5 class="anchor anchorWithStickyNavbar_LWe7" id="the-scverse-project-provides-a-computational-ecosystem-for-single-cell-omics-data-analysis">The scverse project provides a computational ecosystem for single-cell omics data analysis<a href="#the-scverse-project-provides-a-computational-ecosystem-for-single-cell-omics-data-analysis" class="hash-link" aria-label="Direct link to The scverse project provides a computational ecosystem for single-cell omics data analysis" title="Direct link to The scverse project provides a computational ecosystem for single-cell omics data analysis">​</a></h5><p>Virshup+ <em>et al</em>. </p><p><a href="https://www.embopress.org/doi/full/10.15252/msb.202211517" target="_blank" rel="noopener noreferrer">[Nature biotechnology (2023)]</a>,
<a href="https://scverse.org/" target="_blank" rel="noopener noreferrer">[community]</a>.</p><hr><h5 class="anchor anchorWithStickyNavbar_LWe7" id="best-practices-for-single-cell-analysis-across-modalities">Best practices for single-cell analysis across modalities<a href="#best-practices-for-single-cell-analysis-across-modalities" class="hash-link" aria-label="Direct link to Best practices for single-cell analysis across modalities" title="Direct link to Best practices for single-cell analysis across modalities">​</a></h5><p>Heumos+ <em>et al</em>. </p><p><a href="https://www.embopress.org/doi/full/10.15252/msb.202211517" target="_blank" rel="noopener noreferrer">[Nature Reviews Genetics (2023)]</a>,
<a href="https://www.sc-best-practices.org/preamble.html" target="_blank" rel="noopener noreferrer">[code)]</a>.</p><hr><h5 class="anchor anchorWithStickyNavbar_LWe7" id="biologically-informed-deep-learning-to-query-gene-programs-in-single-cell-atlases">Biologically Informed Deep Learning to Query Gene Programs in Single-Cell Atlases<a href="#biologically-informed-deep-learning-to-query-gene-programs-in-single-cell-atlases" class="hash-link" aria-label="Direct link to Biologically Informed Deep Learning to Query Gene Programs in Single-Cell Atlases" title="Direct link to Biologically Informed Deep Learning to Query Gene Programs in Single-Cell Atlases">​</a></h5><u>Lotfollahi M</u>, M+., Rybakov, S+., Hrovatin, K., Hediyeh-Zadeh, S., Talavera-López, C., Misharin, A. V., &amp; Theis, F. J.<p><a href="https://icml-compbio.github.io/icml-website-2021/2021/papers/WCBICML2021_paper_45.pdf" target="_blank" rel="noopener noreferrer">[Nature Cell Biology (2023)]</a>,
<a href="https://github.com/theislab/scArches" target="_blank" rel="noopener noreferrer">[code]</a>.</p><hr><h5 class="anchor anchorWithStickyNavbar_LWe7" id="mapping-cells-to-gene-programs">Mapping cells to gene programs<a href="#mapping-cells-to-gene-programs" class="hash-link" aria-label="Direct link to Mapping cells to gene programs" title="Direct link to Mapping cells to gene programs">​</a></h5><u>Lotfollahi M</u>, M+., Rybakov, S+., Hrovatin, K., Hediyeh-Zadeh, S., Talavera-López, C., Misharin, A. V., &amp; Theis, F. J.<p><a href="https://protocolexchange.researchsquare.com/article/pex-2092/v1" target="_blank" rel="noopener noreferrer">[Protocole exchange (2023)]</a>,
<a href="https://github.com/theislab/scArches" target="_blank" rel="noopener noreferrer">[code]</a>.</p><hr><h4 class="anchor anchorWithStickyNavbar_LWe7" id="single-cell-rna-sequencing-uncovers-heterogeneous-transcriptional-signatures-in-tumor-infiltrated-dendritic-cells-in-prostate-cancer-heliyon-95-e15694">Single-cell RNA sequencing uncovers heterogeneous transcriptional signatures in tumor-infiltrated dendritic cells in prostate cancer. Heliyon, 9(5), e15694.<a href="#single-cell-rna-sequencing-uncovers-heterogeneous-transcriptional-signatures-in-tumor-infiltrated-dendritic-cells-in-prostate-cancer-heliyon-95-e15694" class="hash-link" aria-label="Direct link to Single-cell RNA sequencing uncovers heterogeneous transcriptional signatures in tumor-infiltrated dendritic cells in prostate cancer. Heliyon, 9(5), e15694." title="Direct link to Single-cell RNA sequencing uncovers heterogeneous transcriptional signatures in tumor-infiltrated dendritic cells in prostate cancer. Heliyon, 9(5), e15694.">​</a></h4><p>Feriz, A. M., Khosrojerdi, A., <u>Lotfollahi, M.</u>, Shamsaki, N., GhasemiGol, M., HosseiniGol, E., Fereidouni, M., Rohban, M. H., Sebzari, A. R., Saghafi, S., Leone, P., Silvestris, N., Safarpour, H., &amp; Racanelli, V. (2023). </p><p><a href="https://www.cell.com/heliyon/pdf/S2405-8440(23)02901-8.pdf" target="_blank" rel="noopener noreferrer">[Heliyon(2023)]</a>.</p><h3 class="anchor anchorWithStickyNavbar_LWe7" id="2022">2022<a href="#2022" class="hash-link" aria-label="Direct link to 2022" title="Direct link to 2022">​</a></h3><h5 class="anchor anchorWithStickyNavbar_LWe7" id="predicting-single-cell-perturbation-responses-for-unseen-drugs">Predicting single-cell perturbation responses for unseen drugs<a href="#predicting-single-cell-perturbation-responses-for-unseen-drugs" class="hash-link" aria-label="Direct link to Predicting single-cell perturbation responses for unseen drugs" title="Direct link to Predicting single-cell perturbation responses for unseen drugs">​</a></h5><p>Hetzel, L., Böhm, S., Kilbertus, N., Günnemann, S., <u>Lotfollahi, M.</u>, &amp; Theis, F. (2022).</p><p><a href="https://nips.cc/Conferences/2022/ScheduleMultitrack?event=53227" target="_blank" rel="noopener noreferrer">[NeurIPS (2022)]</a>,
<a href="https://github.com/theislab/chemcpa" target="_blank" rel="noopener noreferrer">[code]</a>.</p><hr><h5 class="anchor anchorWithStickyNavbar_LWe7" id="modelling-method-using-a-conditional-variational-autoencoder">Modelling method using a conditional variational autoencoder<a href="#modelling-method-using-a-conditional-variational-autoencoder" class="hash-link" aria-label="Direct link to Modelling method using a conditional variational autoencoder" title="Direct link to Modelling method using a conditional variational autoencoder">​</a></h5><p>Theis, F. J.,<u>Lotfollahi, M.</u>, Wolf, F. A. </p><p><a href="https://patents.google.com/patent/US20220383985A1/en" target="_blank" rel="noopener noreferrer">US Patent</a>.</p><hr><h5 class="anchor anchorWithStickyNavbar_LWe7" id="deep-learning-in-spatially-resolved-transcriptomics-a-comprehensive-technical-view">Deep Learning in Spatially Resolved Transcriptomics: A Comprehensive Technical View<a href="#deep-learning-in-spatially-resolved-transcriptomics-a-comprehensive-technical-view" class="hash-link" aria-label="Direct link to Deep Learning in Spatially Resolved Transcriptomics: A Comprehensive Technical View" title="Direct link to Deep Learning in Spatially Resolved Transcriptomics: A Comprehensive Technical View">​</a></h5><p>Nasab, R. Z., Ghamsari, M. R. E., Argha, A., Macphillamy, C., Beheshti, A., Alizadehsani, R., Lovell, N. H.,</p><u>Lotfollahi, M.</u>, &amp; Alinejad-Rokny, H.<p><a href="https://arxiv.org/abs/2210.04453" target="_blank" rel="noopener noreferrer">arXiv</a>.</p><hr><h5 class="anchor anchorWithStickyNavbar_LWe7" id="single-cell-reference-mapping-to-construct-and-extend-cell-type-hierarchies">Single-cell reference mapping to construct and extend cell type hierarchies<a href="#single-cell-reference-mapping-to-construct-and-extend-cell-type-hierarchies" class="hash-link" aria-label="Direct link to Single-cell reference mapping to construct and extend cell type hierarchies" title="Direct link to Single-cell reference mapping to construct and extend cell type hierarchies">​</a></h5><p>Michielsen,L+.,<u>Lotfollahi, M.+</u>, Strobl, D., Sikkema, L., Reinders, M. J. T., Theis, F. J.,Mahfouz, A. </p><p><a href="https://www.biorxiv.org/content/10.1101/2022.07.07.499109v1" target="_blank" rel="noopener noreferrer">[BioRxv (2022)]</a>,
<a href="https://github.com/theislab/scarches" target="_blank" rel="noopener noreferrer">[code]</a>.</p><hr><h5 class="anchor anchorWithStickyNavbar_LWe7" id="squidpy-a-scalable-framework-for-spatial-omics-analysis">Squidpy: a scalable framework for spatial omics analysis<a href="#squidpy-a-scalable-framework-for-spatial-omics-analysis" class="hash-link" aria-label="Direct link to Squidpy: a scalable framework for spatial omics analysis" title="Direct link to Squidpy: a scalable framework for spatial omics analysis">​</a></h5><p>Palla, G., Spitzer, H., Klein, M., Fischer, D., Schaar, A. C., Kuemmerle, L. B., Rybakov, S.,
Ibarra, I. L., Holmberg, O., Virshup, I., <u>Lotfollahi, M.</u>, Richter, S. &amp; Theis, F. J. </p><p><a href="https://www.nature.com/articles/s41592-021-01358-2" target="_blank" rel="noopener noreferrer">[Nature Methods (2022)]</a>,
<a href="https://github.com/theislab/squidpy" target="_blank" rel="noopener noreferrer">[code]</a>.</p><hr><h5 class="anchor anchorWithStickyNavbar_LWe7" id="a-python-library-for-probabilistic-analysis-of-single-cell-omics-data">A Python library for probabilistic analysis of single-cell omics data<a href="#a-python-library-for-probabilistic-analysis-of-single-cell-omics-data" class="hash-link" aria-label="Direct link to A Python library for probabilistic analysis of single-cell omics data" title="Direct link to A Python library for probabilistic analysis of single-cell omics data">​</a></h5><p>Gayoso, A.+, Lopez, R.+, Xing, G., Boyeau+, P., Valiollah Pour Amiri, V., Hong, J., Wu, K., Jayasuriya, M., Mehlman, E., Langevin, M., Liu, Y., Samaran, J.,
Misrachi, G., Nazaret, A., Clivio, O., Xu, C., Ashuach, T., Gabitto, M., <u>Lotfollahi, M.</u>, … Yosef, N. </p><p><a href="https://www.nature.com/articles/s41587-021-01206-w" target="_blank" rel="noopener noreferrer">[Nature Biotechnology (2022)]</a>,
<a href="https://scvi-tools.org/" target="_blank" rel="noopener noreferrer">[code]</a>.</p><hr><h5 class="anchor anchorWithStickyNavbar_LWe7" id="population-level-integration-of-single-cell-datasets-enables-multi-scale-analysis-across-samples-1">Population-level integration of single-cell datasets enables multi-scale analysis across samples.<a href="#population-level-integration-of-single-cell-datasets-enables-multi-scale-analysis-across-samples-1" class="hash-link" aria-label="Direct link to Population-level integration of single-cell datasets enables multi-scale analysis across samples." title="Direct link to Population-level integration of single-cell datasets enables multi-scale analysis across samples.">​</a></h5><p>De Donno, C., Hediyeh-Zadeh, S., Wagenstetter, M., Moinfar, A. A., Zappia, L., </p><u>Lotfollahi, M.* </u>, &amp; Theis, F. J*.<p><a href="https://github.com/theislab/scArches" target="_blank" rel="noopener noreferrer">[code]</a>,
<a href="https://www.biorxiv.org/content/10.1101/2022.11.28.517803v1" target="_blank" rel="noopener noreferrer">[bioRxv (2022)]</a>.</p><hr><h5 class="anchor anchorWithStickyNavbar_LWe7" id="an-integrated-cell-atlas-of-the-human-lung-in-health-and-disease-1">An integrated cell atlas of the human lung in health and disease<a href="#an-integrated-cell-atlas-of-the-human-lung-in-health-and-disease-1" class="hash-link" aria-label="Direct link to An integrated cell atlas of the human lung in health and disease" title="Direct link to An integrated cell atlas of the human lung in health and disease">​</a></h5><p>Sikkema, L., Strobl, D., Zappia, L., Madissoon, E., Markov, N. S., Zaragosi, L., Ansari, M., Arguel, M., Apperloo, L., Bécavin, C., Berg, M., Chichelnitskiy, E., Chung, M.,
Collin, A., Gay, A. C. A., Hooshiar Kashani, B., Jain, M., Kapellos, T., Kole, T. M., …,<u>Lotfollahi. M</u>,...,Theis, F.</p><p><a href="https://github.com/LungCellAtlas/HLCA_reproducibility" target="_blank" rel="noopener noreferrer">[code]</a>,</p><p><a href="https://www.biorxiv.org/content/10.1101/2022.03.10.483747v1" target="_blank" rel="noopener noreferrer">[bioRxv (2022)]</a>.</p><hr><h5 class="anchor anchorWithStickyNavbar_LWe7" id="deep-generative-modeling-of-transcriptional-dynamics-for-rna-velocity-analysis-in-single-cells">Deep generative modeling of transcriptional dynamics for RNA velocity analysis in single cells.<a href="#deep-generative-modeling-of-transcriptional-dynamics-for-rna-velocity-analysis-in-single-cells" class="hash-link" aria-label="Direct link to Deep generative modeling of transcriptional dynamics for RNA velocity analysis in single cells." title="Direct link to Deep generative modeling of transcriptional dynamics for RNA velocity analysis in single cells.">​</a></h5><p>Gayoso, A., Weiler, P.,<u> Lotfollahi, M.</u>, Klein, D., Hong, J., Streets, A., Theis, F. J., &amp; Yosef, N.</p><p><a href="https://github.com/YosefLab/velovi" target="_blank" rel="noopener noreferrer">[code]</a>,</p><p><a href="https://www.biorxiv.org/content/10.1101/2022.03.10.483747v1" target="_blank" rel="noopener noreferrer">[bioRxv (2022)]</a>.</p><hr><h5 class="anchor anchorWithStickyNavbar_LWe7" id="multicpa-multimodal-compositional-perturbation-autoencoder">MultiCPA: Multimodal Compositional Perturbation Autoencoder<a href="#multicpa-multimodal-compositional-perturbation-autoencoder" class="hash-link" aria-label="Direct link to MultiCPA: Multimodal Compositional Perturbation Autoencoder" title="Direct link to MultiCPA: Multimodal Compositional Perturbation Autoencoder">​</a></h5><p>Inecik, K., Uhlmann, A.,</p><u>Lotfollahi,M.* </u>, Theis,F*.<p><a href="https://icml-compbio.github.io/icml-website-2022/" target="_blank" rel="noopener noreferrer">[ICML Workshop on Computational Biology (WCB) 2022]</a>,
<a href="https://www.biorxiv.org/content/10.1101/2022.07.08.499049v1" target="_blank" rel="noopener noreferrer">[bioRxv]</a>,
<a href="https://github.com/theislab/multicpa" target="_blank" rel="noopener noreferrer">[code]</a>.</p><hr><h5 class="anchor anchorWithStickyNavbar_LWe7" id="continual-single-cell-architecture-surgery-for-reference-mapping">Continual single-cell architecture surgery for reference mapping.<a href="#continual-single-cell-architecture-surgery-for-reference-mapping" class="hash-link" aria-label="Direct link to Continual single-cell architecture surgery for reference mapping." title="Direct link to Continual single-cell architecture surgery for reference mapping.">​</a></h5><p>Hediyeh-zadeh, S.,</p><u>Lotfollahi, M.* </u>, &amp; Theis, F*.<p><a href="https://icml-compbio.github.io/2022/papers/WCBICML2022_paper_68.pdf" target="_blank" rel="noopener noreferrer">[ICML Workshop on Computational Biology (WCB) 2022]</a>,</p><hr><h5 class="anchor anchorWithStickyNavbar_LWe7" id="mapping-single-cell-data-to-reference-atlases-by-transfer-learning">Mapping single-cell data to reference atlases by transfer learning<a href="#mapping-single-cell-data-to-reference-atlases-by-transfer-learning" class="hash-link" aria-label="Direct link to Mapping single-cell data to reference atlases by transfer learning" title="Direct link to Mapping single-cell data to reference atlases by transfer learning">​</a></h5><u>Lotfollahi, M.</u>, Naghipourfar, M., Luecken, M. D., Khajavi, M., Büttner, M., Wagenstetter, M., Avsec, Ž., Gayoso, A., Yosef, N., Interlandi, M. &amp; Others.<p><a href="https://www.nature.com/articles/s41587-021-01001-7" target="_blank" rel="noopener noreferrer">[Nature Biotechnology (2022)]</a>,
<a href="https://github.com/theislab/scarches" target="_blank" rel="noopener noreferrer">[code]</a>,
<a href="https://www.mdsi.tum.de/en/mdsi/latest-info/news/full-text/article/awardees-of-the-mdsi-best-paper-of-the-year-award/" target="_blank" rel="noopener noreferrer">[MDSI best paper award]</a>,
<a href="https://www.nature.com/nbt/volumes/40/issues/1" target="_blank" rel="noopener noreferrer">[featured cover in Nature Biotechnology]</a>.</p><h3 class="anchor anchorWithStickyNavbar_LWe7" id="2021">2021<a href="#2021" class="hash-link" aria-label="Direct link to 2021" title="Direct link to 2021">​</a></h3><h5 class="anchor anchorWithStickyNavbar_LWe7" id="multigrate-single-cell-multi-omic-data-integration">Multigrate: Single-Cell Multi-Omic Data Integration<a href="#multigrate-single-cell-multi-omic-data-integration" class="hash-link" aria-label="Direct link to Multigrate: Single-Cell Multi-Omic Data Integration" title="Direct link to Multigrate: Single-Cell Multi-Omic Data Integration">​</a></h5><u>Lotfollahi. M+</u>., Litinetskaya, A+. and Theis, F. J.<p><a href="https://icml-compbio.github.io/icml-website-2021/" target="_blank" rel="noopener noreferrer">[contributed talk Award at ICML Workshop on Computational Biology 2021]</a>,
<a href="https://github.com/theislab/multigrate" target="_blank" rel="noopener noreferrer">[code]</a>,
<a href="https://www.biorxiv.org/content/10.1101/2022.03.16.484643v1" target="_blank" rel="noopener noreferrer">[bioRxv (2022)]</a>.</p><hr><h5 class="anchor anchorWithStickyNavbar_LWe7" id="integrating-t-cell-receptor-and-transcriptome-for-large-scale-single-cell-immune-profiling-analysis">Integrating T-cell receptor and transcriptome for large-scale single-cell immune profiling analysis<a href="#integrating-t-cell-receptor-and-transcriptome-for-large-scale-single-cell-immune-profiling-analysis" class="hash-link" aria-label="Direct link to Integrating T-cell receptor and transcriptome for large-scale single-cell immune profiling analysis" title="Direct link to Integrating T-cell receptor and transcriptome for large-scale single-cell immune profiling analysis">​</a></h5><p>Drost, F., An, Y., Dratva, L. M., Lindeboom, R. G. H., Haniffa, M., Teichmann, S. A., Theis, F., <u>Lotfollahi, M.<!-- -->*</u>, Schubert, B<!-- -->*<!-- -->.</p><p><a href="https://icml-compbio.github.io/icml-website-2021/2021/papers/WCBICML2021_paper_45.pdf" target="_blank" rel="noopener noreferrer">[ICML Workshop on Computational Biology 2021]</a>,
<a href="https://github.com/SchubertLab/mvTCR" target="_blank" rel="noopener noreferrer">[code]</a>,
<a href="https://www.biorxiv.org/content/10.1101/2021.06.24.449733v2.abstract" target="_blank" rel="noopener noreferrer">[bioRxv (2021)]</a>.</p><hr><h5 class="anchor anchorWithStickyNavbar_LWe7" id="machine-learning-for-perturbational-single-cell-omics">Machine learning for perturbational single-cell omics<a href="#machine-learning-for-perturbational-single-cell-omics" class="hash-link" aria-label="Direct link to Machine learning for perturbational single-cell omics" title="Direct link to Machine learning for perturbational single-cell omics">​</a></h5><p>Ji, Y.,<u> Lotfollahi, M.</u>, Wolf, F. A. &amp; Theis, F. J.</p><p><a href="https://doi.org/10.1016/j.cels.2021.05.016" target="_blank" rel="noopener noreferrer">[Cell Systems(2021)]</a>,
<a href="https://github.com/theislab/sc-pert" target="_blank" rel="noopener noreferrer">[data resource]</a>.</p><h3 class="anchor anchorWithStickyNavbar_LWe7" id="2020">2020<a href="#2020" class="hash-link" aria-label="Direct link to 2020" title="Direct link to 2020">​</a></h3><h5 class="anchor anchorWithStickyNavbar_LWe7" id="out-of-distribution-prediction-with-disentangled-representations-for-single-cell-rna-sequencing-data">Out-of-distribution prediction with disentangled representations for single-cell RNA sequencing data<a href="#out-of-distribution-prediction-with-disentangled-representations-for-single-cell-rna-sequencing-data" class="hash-link" aria-label="Direct link to Out-of-distribution prediction with disentangled representations for single-cell RNA sequencing data" title="Direct link to Out-of-distribution prediction with disentangled representations for single-cell RNA sequencing data">​</a></h5><u>Lotfollahi, M.+</u>, Dony, L.+, Agarwala, H.+, &amp; Theis, F. J.<p><a href="https://icml-compbio.github.io/icml-website-2020/" target="_blank" rel="noopener noreferrer">[ICML Workshop on Computational Biology (WCB) 2020]</a>,
<a href="https://slideslive.com/38931310/outofdistribution-prediction-with-disentangled-representations-for-scrnaseq-data" target="_blank" rel="noopener noreferrer">[spotlight talk ICML WCB 2020]</a>,
<a href="https://github.com/theislab/disent" target="_blank" rel="noopener noreferrer">[code]</a></p><hr><h4 class="anchor anchorWithStickyNavbar_LWe7" id="conditional-out-of-distribution-generation-for-unpaired-data-using-transfer-vae">Conditional out-of-distribution generation for unpaired data using transfer VAE<a href="#conditional-out-of-distribution-generation-for-unpaired-data-using-transfer-vae" class="hash-link" aria-label="Direct link to Conditional out-of-distribution generation for unpaired data using transfer VAE" title="Direct link to Conditional out-of-distribution generation for unpaired data using transfer VAE">​</a></h4><u>Lotfollahi, M.</u>, Naghipourfar, M., Theis, F. J. &amp; Wolf, F. A.<p><a href="https://doi.org/10.1093/bioinformatics/btaa800" target="_blank" rel="noopener noreferrer">[Bioinformatics(2020)]</a>,
<a href="https://github.com/theislab/trvae" target="_blank" rel="noopener noreferrer">[<!-- -->code<!-- -->]</a>,
<a href="https://eccb2020.info/" target="_blank" rel="noopener noreferrer">[<!-- -->talk at ECCB 2020 (21.18% accept rate)<!-- -->]</a>.</p><hr><p><strong>Learning Interpretable Latent Autoencoder Representations with Annotations of Feature Sets</strong></p><p>S. Rybakov, <u>M. Lotfollahi</u>, F.J. Theis<em>, F.A. Wolf</em>.</p><p><a href="https://doi.org/10.1101/2020.12.02.401182" target="_blank" rel="noopener noreferrer">[Machine Learning in Computational Biology (2020)]</a>,
<a href="https://github.com/theislab/intercode" target="_blank" rel="noopener noreferrer">[code]</a>.</p><h3 class="anchor anchorWithStickyNavbar_LWe7" id="2019">2019<a href="#2019" class="hash-link" aria-label="Direct link to 2019" title="Direct link to 2019">​</a></h3><p><strong>scGen predicts single-cell perturbation responses.</strong></p><u>Lotfollahi, M.</u>, Wolf, F. A. &amp; Theis, F. J.<p><a href="https://doi.org/10.1038/s41592-019-0494-8" target="_blank" rel="noopener noreferrer">[<!-- -->Nature Methods(2019)<!-- -->]</a>,
<a href="https://github.com/theislab/scgen" target="_blank" rel="noopener noreferrer">[<!-- -->code<!-- -->]</a>,
<a href="https://www.bioengineering.tum.de/en/news/details/ai-extrapolates-from-mice-to-humans-1" target="_blank" rel="noopener noreferrer">[<!-- -->press<!-- -->]</a>.</p><hr><h5 class="anchor anchorWithStickyNavbar_LWe7" id="deep-packet-a-novel-approach-for-encrypted-traffic-classification-using-deep-learning">Deep packet: a novel approach for encrypted traffic classification using deep learning<a href="#deep-packet-a-novel-approach-for-encrypted-traffic-classification-using-deep-learning" class="hash-link" aria-label="Direct link to Deep packet: a novel approach for encrypted traffic classification using deep learning" title="Direct link to Deep packet: a novel approach for encrypted traffic classification using deep learning">​</a></h5><u>Lotfollahi, M.</u>, Jafari Siavoshani, M., Shirali Hossein Zade, R., &amp; Saberian, M.<p><a href="https://link.springer.com/article/10.1007/s00500-019-04030-2?wt_mc=Internal.Event.1.SEM.ArticleAuthorOnlineFirst&amp;utm_source=ArticleAuthorOnlineFirst&amp;utm_medium=email&amp;utm_content=AA_en_06082018&amp;ArticleAuthorOnlineFirst_20190516&amp;error=cookies_not_supported&amp;code=04315f4e-0ae2-48fb-9e2f-c2d25db1a332" target="_blank" rel="noopener noreferrer">[<!-- -->soft computing(2019)<!-- -->]</a>.</p></article></div><div class="col col--2"><div class="tableOfContents_bqdL thin-scrollbar"><ul class="table-of-contents table-of-contents__left-border"><li><a href="#2024" class="table-of-contents__link toc-highlight">2024</a></li><li><a href="#2023" class="table-of-contents__link toc-highlight">2023</a></li><li><a href="#2022" class="table-of-contents__link toc-highlight">2022</a></li><li><a href="#2021" class="table-of-contents__link toc-highlight">2021</a></li><li><a href="#2020" class="table-of-contents__link toc-highlight">2020</a></li><li><a href="#2019" class="table-of-contents__link toc-highlight">2019</a></li></ul></div></div></div></main></div><footer class="footer"><div class="container container-fluid"><div class="footer__bottom text--center"><div class="footer__copyright">Copyright © 2024 Mo Lotfollahi.</div></div></div></footer></div>
<script src="/assets/js/runtime~main.49356ac3.js"></script>
<script src="/assets/js/main.52fb0cbd.js"></script>
</body>
</html>